<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Summarize contents of a Neo4j KG engine — summary.neo4j_engine • monarchr</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Summarize contents of a Neo4j KG engine — summary.neo4j_engine"><meta property="og:description" content="Given a Neo4j based KG engine, provides summary information in the form of
node counts, category counts across nodes, relationship type counts, and available properties.
General information about the graph is printed to the console, and a list of
dataframes with this information is returned invisibly. Also returned
are cats, preds, and props entries, containing lists of available
categories/predicates/properties for convenient auto-completion in RStudio."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">


    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">monarchr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">2.1.2</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/monarchr.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li class="divider">
    <li class="dropdown-header">Examples</li>
    <li>
      <a href="../articles/examples/visualizing_kgs.html">Visualizing Knowledge Graphs</a>
    </li>
    <li>
      <a href="../articles/examples/exploring_kgs.html">Exploring Knowledge Graphs</a>
    </li>
    <li>
      <a href="../articles/examples/transitive_rollups.html">Rollups and Transitivity</a>
    </li>
    <li>
      <a href="../articles/examples/engine_preferences.html">Engine Preferences</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/monarch-initiative/monarchr/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Summarize contents of a Neo4j KG engine</h1>
    <small class="dont-index">Source: <a href="https://github.com/monarch-initiative/monarchr/blob/plotfix/R/summary.neo4j_engine.R" class="external-link"><code>R/summary.neo4j_engine.R</code></a></small>
    <div class="hidden name"><code>summary.neo4j_engine.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Given a Neo4j based KG engine, provides summary information in the form of
node counts, category counts across nodes, relationship type counts, and available properties.
General information about the graph is printed to the console, and a list of
dataframes with this information is returned invisibly. Also returned
are <code>cats</code>, <code>preds</code>, and <code>props</code> entries, containing lists of available
categories/predicates/properties for convenient auto-completion in RStudio.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="co"># S3 method for class 'neo4j_engine'</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">object</span>, <span class="va">...</span>, quiet <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>


<dl><dt id="arg-object">object<a class="anchor" aria-label="anchor" href="#arg-object"></a></dt>
<dd><p>A <code>neo4j_engine</code> object</p></dd>


<dt id="arg--">...<a class="anchor" aria-label="anchor" href="#arg--"></a></dt>
<dd><p>Other parameters (not used)</p></dd>


<dt id="arg-quiet">quiet<a class="anchor" aria-label="anchor" href="#arg-quiet"></a></dt>
<dd><p>Logical, whether to suppress printing of the summary</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    <p>A list of dataframes and named lists</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># prints a readable summary and returns a list of dataframes</span></span></span>
<span class="r-in"><span><span class="va">stats</span> <span class="op">&lt;-</span> <span class="fu"><a href="monarch_engine.html">monarch_engine</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Trying to connect to https://neo4j.monarchinitiative.org</span>
<span class="r-msg co"><span class="r-pr">#&gt;</span> Connected to https://neo4j.monarchinitiative.org</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> A Neo4j-backed knowledge graph engine.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Gathering statistics, please wait...</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Total nodes:  1346185 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Total edges:  14621271 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Node category counts:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                   category  count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               biolink:Gene 581682</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>    biolink:SequenceVariant 209865</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:PhenotypicFeature 169316</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           biolink:Genotype 137639</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>   biolink:AnatomicalEntity  66029</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:BiologicalProcess  30769</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:Disease  30384</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>         biolink:NamedThing  27656</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:Protein  25098</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:Pathway  22254</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     biolink:ChemicalEntity  22231</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:MolecularActivity  12723</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:CellularComponent   4538</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>               biolink:Cell   3171</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>      biolink:OrganismTaxon   1987</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>    biolink:MolecularEntity    529</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>          biolink:LifeStage    314</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Edge type counts:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                           predicate   count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                              biolink:interacts_with 2797006</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                biolink:expressed_in 2379934</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                               biolink:has_phenotype 2077069</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                              biolink:orthologous_to 1506263</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                     biolink:enables 1239419</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                        biolink:actively_involved_in 1103592</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                  biolink:located_in  994683</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                 biolink:subclass_of  612920</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                  biolink:related_to  520067</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                             biolink:participates_in  329190</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                  biolink:acts_upstream_of_or_within  295324</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                biolink:is_active_in  188945</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                      biolink:is_sequence_variant_of  181003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                        biolink:has_sequence_variant  157143</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                     biolink:part_of  129977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>       biolink:treats_or_applied_or_studied_to_treat   18934</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                      biolink:causes   18203</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                            biolink:acts_upstream_of   17896</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                    biolink:model_of   10069</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                              biolink:contributes_to    9837</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                     biolink:has_mode_of_inheritance    8829</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>              biolink:gene_associated_with_condition    8089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     biolink:associated_with_increased_likelihood_of    4394</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                            biolink:colocalizes_with    4251</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                 biolink:genetically_associated_with    3484</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                        biolink:disease_has_location    1020</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:acts_upstream_of_positive_effect     992</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:acts_upstream_of_or_within_positive_effect     826</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                       biolink:ameliorates_condition     786</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:acts_upstream_of_negative_effect     336</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:acts_upstream_of_or_within_negative_effect     279</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                             biolink:has_participant     269</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                  biolink:preventative_for_condition     145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                    biolink:disrupts      89</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                   biolink:caused_by       5</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                          biolink:contraindicated_in       3</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Available node and edge properties:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [1] "category"                         "name"                            </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [3] "provided_by"                      "in_taxon"                        </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [5] "in_taxon_label"                   "namespace"                       </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [7] "id"                               "description"                     </span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  [9] "synonym"                          "exact_synonyms"                  </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [11] "iri"                              "deprecated"                      </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [13] "full_name"                        "symbol"                          </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [15] "xref"                             "type"                            </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [17] "narrow_synonyms"                  "broad_synonyms"                  </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [19] "related_synonyms"                 "subsets"                         </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [21] "has_gene"                         "predicate"                       </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [23] "agent_type"                       "aggregator_knowledge_source"     </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [25] "knowledge_level"                  "primary_knowledge_source"        </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [27] "publications"                     "subject"                         </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [29] "object"                           "original_subject"                </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [31] "original_object"                  "has_evidence"                    </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [33] "disease_context_qualifier"        "has_count"                       </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [35] "has_percentage"                   "has_quotient"                    </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [37] "has_total"                        "frequency_qualifier"             </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [39] "qualifiers"                       "stage_qualifier"                 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [41] "original_predicate"               "subject_specialization_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [43] "qualifier"                        "object_specialization_qualifier" </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [45] "negated"                          "onset_qualifier"                 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [47] "sex_qualifier"                    "species_context_qualifier"       </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> For more information about Biolink node (Class) and edge (Association) properties, see https://biolink.github.io/biolink-model/.</span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">stats</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $node_summary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                     category  count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1               biolink:Gene 581682</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2    biolink:SequenceVariant 209865</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3  biolink:PhenotypicFeature 169316</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4           biolink:Genotype 137639</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5   biolink:AnatomicalEntity  66029</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6  biolink:BiologicalProcess  30769</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 7            biolink:Disease  30384</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 8         biolink:NamedThing  27656</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 9            biolink:Protein  25098</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 10           biolink:Pathway  22254</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 11    biolink:ChemicalEntity  22231</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 12 biolink:MolecularActivity  12723</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 13 biolink:CellularComponent   4538</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 14              biolink:Cell   3171</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 15     biolink:OrganismTaxon   1987</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 16   biolink:MolecularEntity    529</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 17         biolink:LifeStage    314</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $edge_summary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                             predicate   count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1                              biolink:interacts_with 2797006</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2                                biolink:expressed_in 2379934</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3                               biolink:has_phenotype 2077069</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4                              biolink:orthologous_to 1506263</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5                                     biolink:enables 1239419</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6                        biolink:actively_involved_in 1103592</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 7                                  biolink:located_in  994683</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 8                                 biolink:subclass_of  612920</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 9                                  biolink:related_to  520067</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 10                            biolink:participates_in  329190</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 11                 biolink:acts_upstream_of_or_within  295324</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 12                               biolink:is_active_in  188945</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 13                     biolink:is_sequence_variant_of  181003</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 14                       biolink:has_sequence_variant  157143</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 15                                    biolink:part_of  129977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 16      biolink:treats_or_applied_or_studied_to_treat   18934</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 17                                     biolink:causes   18203</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 18                           biolink:acts_upstream_of   17896</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 19                                   biolink:model_of   10069</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 20                             biolink:contributes_to    9837</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 21                    biolink:has_mode_of_inheritance    8829</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 22             biolink:gene_associated_with_condition    8089</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 23    biolink:associated_with_increased_likelihood_of    4394</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 24                           biolink:colocalizes_with    4251</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 25                biolink:genetically_associated_with    3484</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 26                       biolink:disease_has_location    1020</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 27           biolink:acts_upstream_of_positive_effect     992</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 28 biolink:acts_upstream_of_or_within_positive_effect     826</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 29                      biolink:ameliorates_condition     786</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 30           biolink:acts_upstream_of_negative_effect     336</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 31 biolink:acts_upstream_of_or_within_negative_effect     279</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 32                            biolink:has_participant     269</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 33                 biolink:preventative_for_condition     145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 34                                   biolink:disrupts      89</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 35                                  biolink:caused_by       5</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 36                         biolink:contraindicated_in       3</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $total_nodes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] 1346185</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $total_edges</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] 14621271</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Gene`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Gene"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:SequenceVariant`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:SequenceVariant"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:PhenotypicFeature`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:PhenotypicFeature"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Genotype`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Genotype"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:AnatomicalEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:AnatomicalEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:BiologicalProcess`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:BiologicalProcess"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Disease`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Disease"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:NamedThing`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:NamedThing"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Protein`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Protein"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Pathway`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Pathway"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ChemicalEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ChemicalEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:MolecularActivity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:MolecularActivity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:CellularComponent`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:CellularComponent"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Cell`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Cell"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:OrganismTaxon`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:OrganismTaxon"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:MolecularEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:MolecularEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:LifeStage`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:LifeStage"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:interacts_with`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:interacts_with"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:expressed_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:expressed_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_phenotype`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_phenotype"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:orthologous_to`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:orthologous_to"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:enables`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:enables"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:actively_involved_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:actively_involved_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:located_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:located_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:subclass_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:subclass_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:related_to`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:related_to"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:participates_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:participates_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of_or_within`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of_or_within"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:is_active_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:is_active_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:is_sequence_variant_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:is_sequence_variant_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_sequence_variant`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_sequence_variant"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:part_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:part_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:treats_or_applied_or_studied_to_treat`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:treats_or_applied_or_studied_to_treat"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:causes`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:causes"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:model_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:model_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:contributes_to`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:contributes_to"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_mode_of_inheritance`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_mode_of_inheritance"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:gene_associated_with_condition`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:gene_associated_with_condition"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:associated_with_increased_likelihood_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:associated_with_increased_likelihood_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:colocalizes_with`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:colocalizes_with"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:genetically_associated_with`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:genetically_associated_with"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:disease_has_location`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:disease_has_location"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of_positive_effect`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of_positive_effect"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of_or_within_positive_effect`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of_or_within_positive_effect"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:ameliorates_condition`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ameliorates_condition"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of_negative_effect`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of_negative_effect"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:acts_upstream_of_or_within_negative_effect`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:acts_upstream_of_or_within_negative_effect"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_participant`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_participant"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:preventative_for_condition`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:preventative_for_condition"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:disrupts`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:disrupts"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:caused_by`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:caused_by"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:contraindicated_in`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:contraindicated_in"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$category</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "category"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$name</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "name"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$provided_by</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "provided_by"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$in_taxon</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "in_taxon"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$in_taxon_label</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "in_taxon_label"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$namespace</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "namespace"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$id</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "id"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$description</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "description"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$synonym</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "synonym"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$exact_synonyms</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "exact_synonyms"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$iri</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "iri"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$deprecated</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "deprecated"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$full_name</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "full_name"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$symbol</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "symbol"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$xref</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "xref"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$type</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "type"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$narrow_synonyms</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "narrow_synonyms"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$broad_synonyms</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "broad_synonyms"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$related_synonyms</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "related_synonyms"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$subsets</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "subsets"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_gene</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_gene"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$predicate</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "predicate"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$agent_type</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "agent_type"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$aggregator_knowledge_source</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "aggregator_knowledge_source"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$knowledge_level</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "knowledge_level"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$primary_knowledge_source</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "primary_knowledge_source"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$publications</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "publications"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$subject</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "subject"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$object</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "object"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$original_subject</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "original_subject"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$original_object</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "original_object"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_evidence</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_evidence"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$disease_context_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "disease_context_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_count"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_percentage</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_percentage"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_quotient</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_quotient"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$has_total</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "has_total"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$frequency_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "frequency_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$qualifiers</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "qualifiers"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$stage_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "stage_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$original_predicate</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "original_predicate"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$subject_specialization_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "subject_specialization_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$object_specialization_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "object_specialization_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$negated</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "negated"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$onset_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "onset_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$sex_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "sex_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $props$species_context_qualifier</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "species_context_qualifier"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Shawn O’Neil, Brian Schilder.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.2.0.</p>
</div>

      </footer></div>






  </body></html>

