<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8"><meta charset="utf-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Summarize contents of a KGX-file-based KG engine — summary.file_engine • monarchr</title><!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous"><script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css"><script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous"><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous"><!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet"><script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet"><meta property="og:title" content="Summarize contents of a KGX-file-based KG engine — summary.file_engine"><meta property="og:description" content="Given a KGX file-based KG engine, provides summary information in the form of
node counts, category counts across nodes, and relationship type counts.
General information about the graph is printed to the console, and a list of
dataframes describing node and edge counts is returned invisibly. Also returned
are cats and preds entries, containing lists of available node categories and
edge predicates, respectively, for convenient auto-completion in RStudio."><!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]--></head><body data-spy="scroll" data-target="#toc">


    <div class="container template-reference-topic">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">monarchr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">2.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav"><li>
  <a href="../articles/monarchr.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu"><li class="divider">
    <li class="dropdown-header">Examples</li>
    <li>
      <a href="../articles/examples/visualizing_kgs.html">Visualizing Knowledge Graphs</a>
    </li>
    <li>
      <a href="../articles/examples/exploring_kgs.html">Exploring Knowledge Graphs</a>
    </li>
    <li>
      <a href="../articles/examples/transitive_rollups.html">Rollups and Transitivity</a>
    </li>
  </ul></li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul><ul class="nav navbar-nav navbar-right"><li>
  <a href="https://github.com/monarch-initiative/monarchr/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul></div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->



      </header><div class="row">
  <div class="col-md-9 contents">
    <div class="page-header">
    <h1>Summarize contents of a KGX-file-based KG engine</h1>
    <small class="dont-index">Source: <a href="https://github.com/monarch-initiative/monarchr/blob/main/R/summary.file_engine.R" class="external-link"><code>R/summary.file_engine.R</code></a></small>
    <div class="hidden name"><code>summary.file_engine.Rd</code></div>
    </div>

    <div class="ref-description">
    <p>Given a KGX file-based KG engine, provides summary information in the form of
node counts, category counts across nodes, and relationship type counts.
General information about the graph is printed to the console, and a list of
dataframes describing node and edge counts is returned invisibly. Also returned
are <code>cats</code> and <code>preds</code> entries, containing lists of available node categories and
edge predicates, respectively, for convenient auto-completion in RStudio.</p>
    </div>

    <div id="ref-usage">
    <div class="sourceCode"><pre class="sourceCode r"><code><span><span class="co"># S3 method for class 'file_engine'</span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">object</span>, <span class="va">...</span>, quiet <span class="op">=</span> <span class="cn">FALSE</span><span class="op">)</span></span></code></pre></div>
    </div>

    <div id="arguments">
    <h2>Arguments</h2>


<dl><dt id="arg-object">object<a class="anchor" aria-label="anchor" href="#arg-object"></a></dt>
<dd><p>A <code>file_engine</code> object</p></dd>


<dt id="arg--">...<a class="anchor" aria-label="anchor" href="#arg--"></a></dt>
<dd><p>Other parameters (not used)</p></dd>


<dt id="arg-quiet">quiet<a class="anchor" aria-label="anchor" href="#arg-quiet"></a></dt>
<dd><p>Logical, whether to suppress printing of the summary</p></dd>

</dl></div>
    <div id="value">
    <h2>Value</h2>
    <p>A list of dataframes and named lists</p>
    </div>

    <div id="ref-examples">
    <h2>Examples</h2>
    <div class="sourceCode"><pre class="sourceCode r"><code><span class="r-in"><span><span class="co"># Using example KGX file packaged with monarchr</span></span></span>
<span class="r-in"><span><span class="va">filename</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"eds_marfan_kg.tar.gz"</span>, package <span class="op">=</span> <span class="st">"monarchr"</span><span class="op">)</span></span></span>
<span class="r-in"><span><span class="co"># prints a readable summary and returns a list of dataframes</span></span></span>
<span class="r-in"><span><span class="va">res</span> <span class="op">&lt;-</span> <span class="fu"><a href="file_engine.html">file_engine</a></span><span class="op">(</span><span class="va">filename</span><span class="op">)</span> <span class="op">|&gt;</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> A KGX file-backed knowledge graph engine.</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Total nodes:  3000 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Total edges:  7148 </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Node category counts:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                      category count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                biolink:Entity  3000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                            biolink:NamedThing  3000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                      biolink:BiologicalEntity  2977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                        biolink:ThingWithTaxon  2977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:DiseaseOrPhenotypicFeature  2846</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                     biolink:PhenotypicFeature  2736</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                       biolink:PhysicalEssence   145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:PhysicalEssenceOrOccurrent   145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                         biolink:GenomicEntity   131</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                         biolink:OntologyClass   131</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                               biolink:Disease   110</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                       biolink:SequenceVariant    81</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>     biolink:ChemicalEntityOrGeneOrGeneProduct    37</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                              biolink:Genotype    27</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                  biolink:Gene    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                     biolink:GeneOrGeneProduct    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>            biolink:MacromolecularMachineMixin    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                        biolink:ChemicalEntity    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:ChemicalEntityOrProteinOrPolypeptide    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>             biolink:ChemicalOrDrugOrTreatment    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                       biolink:MolecularEntity    13</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> Edge type counts:</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                        predicate count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                              biolink:subclass_of  5244</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                            biolink:has_phenotype  1709</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                   biolink:causes    56</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>  biolink:associated_with_increased_likelihood_of    38</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>           biolink:gene_associated_with_condition    28</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                 biolink:model_of    27</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                  biolink:has_mode_of_inheritance    26</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>              biolink:genetically_associated_with    11</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                               biolink:related_to     8</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>    biolink:treats_or_applied_or_studied_to_treat     1</span>
<span class="r-in"><span><span class="fu"><a href="https://rdrr.io/r/base/print.html" class="external-link">print</a></span><span class="op">(</span><span class="va">res</span><span class="op">)</span></span></span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $node_summary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                        category count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1                                biolink:Entity  3000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2                            biolink:NamedThing  3000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3                      biolink:BiologicalEntity  2977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4                        biolink:ThingWithTaxon  2977</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5            biolink:DiseaseOrPhenotypicFeature  2846</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6                     biolink:PhenotypicFeature  2736</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 7                       biolink:PhysicalEssence   145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 8            biolink:PhysicalEssenceOrOccurrent   145</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 9                         biolink:GenomicEntity   131</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 10                        biolink:OntologyClass   131</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 11                              biolink:Disease   110</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 12                      biolink:SequenceVariant    81</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 13    biolink:ChemicalEntityOrGeneOrGeneProduct    37</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 14                             biolink:Genotype    27</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 15                                 biolink:Gene    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 16                    biolink:GeneOrGeneProduct    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 17           biolink:MacromolecularMachineMixin    23</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 18                       biolink:ChemicalEntity    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 19 biolink:ChemicalEntityOrProteinOrPolypeptide    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 20            biolink:ChemicalOrDrugOrTreatment    14</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 21                      biolink:MolecularEntity    13</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $edge_summary</span>
<span class="r-out co"><span class="r-pr">#&gt;</span>                                          predicate count</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 1                              biolink:subclass_of  5244</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 2                            biolink:has_phenotype  1709</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 3                                   biolink:causes    56</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 4  biolink:associated_with_increased_likelihood_of    38</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 5           biolink:gene_associated_with_condition    28</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 6                                 biolink:model_of    27</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 7                  biolink:has_mode_of_inheritance    26</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 8              biolink:genetically_associated_with    11</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 9                               biolink:related_to     8</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> 10   biolink:treats_or_applied_or_studied_to_treat     1</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $total_nodes</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] 3000</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $total_edges</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] 7148</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Entity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Entity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:NamedThing`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:NamedThing"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:BiologicalEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:BiologicalEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ThingWithTaxon`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ThingWithTaxon"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:DiseaseOrPhenotypicFeature`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:DiseaseOrPhenotypicFeature"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:PhenotypicFeature`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:PhenotypicFeature"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:PhysicalEssence`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:PhysicalEssence"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:PhysicalEssenceOrOccurrent`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:PhysicalEssenceOrOccurrent"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:GenomicEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:GenomicEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:OntologyClass`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:OntologyClass"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Disease`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Disease"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:SequenceVariant`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:SequenceVariant"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ChemicalEntityOrGeneOrGeneProduct`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ChemicalEntityOrGeneOrGeneProduct"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Genotype`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Genotype"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:Gene`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:Gene"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:GeneOrGeneProduct`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:GeneOrGeneProduct"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:MacromolecularMachineMixin`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:MacromolecularMachineMixin"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ChemicalEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ChemicalEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ChemicalEntityOrProteinOrPolypeptide`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ChemicalEntityOrProteinOrPolypeptide"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:ChemicalOrDrugOrTreatment`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:ChemicalOrDrugOrTreatment"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $cats$`biolink:MolecularEntity`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:MolecularEntity"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:subclass_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:subclass_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_phenotype`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_phenotype"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:causes`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:causes"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:associated_with_increased_likelihood_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:associated_with_increased_likelihood_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:gene_associated_with_condition`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:gene_associated_with_condition"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:model_of`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:model_of"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:has_mode_of_inheritance`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:has_mode_of_inheritance"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:genetically_associated_with`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:genetically_associated_with"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:related_to`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:related_to"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> $preds$`biolink:treats_or_applied_or_studied_to_treat`</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> [1] "biolink:treats_or_applied_or_studied_to_treat"</span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
<span class="r-out co"><span class="r-pr">#&gt;</span> </span>
</code></pre></div>
    </div>
  </div>
  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <nav id="toc" data-toggle="toc" class="sticky-top"><h2 data-toc-skip>Contents</h2>
    </nav></div>
</div>


      <footer><div class="copyright">
  <p></p><p>Developed by Shawn O’Neil, Brian Schilder.</p>
</div>

<div class="pkgdown">
  <p></p><p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.</p>
</div>

      </footer></div>






  </body></html>

