<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Exploring Knowledge Graphs • monarchr</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../../bootstrap-toc.css">
<script src="../../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../../pkgdown.css" rel="stylesheet">
<script src="../../pkgdown.js"></script><link href="../../extra.css" rel="stylesheet">
<meta property="og:title" content="Exploring Knowledge Graphs">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../../index.html">monarchr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">2.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../../articles/monarchr.html">Get started</a>
</li>
<li>
  <a href="../../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li class="divider">
    </li>
<li class="dropdown-header">Examples</li>
    <li>
      <a href="../../articles/examples/visualizing_kgs.html">Visualizing Knowledge Graphs</a>
    </li>
    <li>
      <a href="../../articles/examples/exploring_kgs.html">Exploring Knowledge Graphs</a>
    </li>
    <li>
      <a href="../../articles/examples/transitive_rollups.html">Rollups and Transitivity</a>
    </li>
  </ul>
</li>
<li>
  <a href="../../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/monarch-initiative/monarchr/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="exploring_kgs_files/kePrint-0.0.1/kePrint.js"></script><link href="exploring_kgs_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>Exploring Knowledge Graphs</h1>
                        <h4 data-toc-skip class="author">Shawn T
O’Neil</h4>
            
            <h4 data-toc-skip class="date">Vignette updated:
<i>Jun-04-2025</i>
</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/monarch-initiative/monarchr/blob/1_7_bugfix/vignettes/examples/exploring_kgs.Rmd" class="external-link"><code>vignettes/examples/exploring_kgs.Rmd</code></a></small>
      <div class="hidden name"><code>exploring_kgs.Rmd</code></div>

    </div>

    
    
<p>Knowledge Graphs (KGs) may contain large amounts of information; the
Monarch Initiative KG for example contains not only millions of nodes
and edges, but each node may belong to one or more <em>categories</em>,
across dozens of available categories. While edges may only have a
single <em>predicate</em> linking a subject and object node, there are
similarly dozens of available predicates. Each node category and edge
predicate may further come with other node or edge properties, and these
may be shared across some but not all node categories or edge
predicates.</p>
<p>To help navigate this extensive information, <code>monarchr</code>
provides two functions that may be applied to KG engines: a
<code><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary()</a></code> function that counts these categories and
predicates across nodes and edges, and an <code><a href="../../reference/example_graph.html">example_graph()</a></code>
function that returns a (non-random) subgraph gauranteed to represent
every node category and edge predicate.</p>
<p>As usual, we being by loading the <code>monarchr</code> package,
along with <code>tidygraph</code> and <code>dplyr</code> which tend to
be useful (but we will not actually use in this vignette).</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://github.com/monarch-initiative/monarchr" class="external-link">monarchr</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://tidygraph.data-imaginist.com" class="external-link">tidygraph</a></span><span class="op">)</span></span>
<span><span class="kw"><a href="https://rdrr.io/r/base/library.html" class="external-link">library</a></span><span class="op">(</span><span class="va"><a href="https://dplyr.tidyverse.org" class="external-link">dplyr</a></span><span class="op">)</span></span></code></pre></div>
<div class="section level2">
<h2 id="engine-summary">Engine <code>summary()</code><a class="anchor" aria-label="anchor" href="#engine-summary"></a>
</h2>
<p>The summary function, when applied to a KG engine (like
<code><a href="../../reference/file_engine.html">file_engine()</a></code>, <code><a href="../../reference/neo4j_engine.html">neo4j_engine()</a></code>, or the
cloud-hosted <code><a href="../../reference/monarch_engine.html">monarch_engine()</a></code>), prints counts of nodes and
edges broken out by available node category and edge predicate. To keep
the information small, we’ll produce a summary of the included mini-KG
containing information about Ehlers-Danlos Syndrome (EDS) and Marfan
Syndrome:</p>
<div class="sourceCode" id="cb2"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">filename</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"eds_marfan_kg.tar.gz"</span>, package <span class="op">=</span> <span class="st">"monarchr"</span><span class="op">)</span></span>
<span><span class="va">eds_marfan</span> <span class="op">&lt;-</span> <span class="fu"><a href="../../reference/file_engine.html">file_engine</a></span><span class="op">(</span><span class="va">filename</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">eds_marfan</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## </span></span>
<span><span class="co">## A KGX file-backed knowledge graph engine.</span></span>
<span><span class="co">## Total nodes:  3000 </span></span>
<span><span class="co">## Total edges:  7148 </span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Node category counts:</span></span>
<span><span class="co">##                                      category count</span></span>
<span><span class="co">##                                biolink:Entity  3000</span></span>
<span><span class="co">##                            biolink:NamedThing  3000</span></span>
<span><span class="co">##                      biolink:BiologicalEntity  2977</span></span>
<span><span class="co">##                        biolink:ThingWithTaxon  2977</span></span>
<span><span class="co">##            biolink:DiseaseOrPhenotypicFeature  2846</span></span>
<span><span class="co">##                     biolink:PhenotypicFeature  2736</span></span>
<span><span class="co">##                       biolink:PhysicalEssence   145</span></span>
<span><span class="co">##            biolink:PhysicalEssenceOrOccurrent   145</span></span>
<span><span class="co">##                         biolink:GenomicEntity   131</span></span>
<span><span class="co">##                         biolink:OntologyClass   131</span></span>
<span><span class="co">##                               biolink:Disease   110</span></span>
<span><span class="co">##                       biolink:SequenceVariant    81</span></span>
<span><span class="co">##     biolink:ChemicalEntityOrGeneOrGeneProduct    37</span></span>
<span><span class="co">##                              biolink:Genotype    27</span></span>
<span><span class="co">##                                  biolink:Gene    23</span></span>
<span><span class="co">##                     biolink:GeneOrGeneProduct    23</span></span>
<span><span class="co">##            biolink:MacromolecularMachineMixin    23</span></span>
<span><span class="co">##                        biolink:ChemicalEntity    14</span></span>
<span><span class="co">##  biolink:ChemicalEntityOrProteinOrPolypeptide    14</span></span>
<span><span class="co">##             biolink:ChemicalOrDrugOrTreatment    14</span></span>
<span><span class="co">##                       biolink:MolecularEntity    13</span></span>
<span><span class="co">## </span></span>
<span><span class="co">## Edge type counts:</span></span>
<span><span class="co">##                                        predicate count</span></span>
<span><span class="co">##                              biolink:subclass_of  5244</span></span>
<span><span class="co">##                            biolink:has_phenotype  1709</span></span>
<span><span class="co">##                                   biolink:causes    56</span></span>
<span><span class="co">##  biolink:associated_with_increased_likelihood_of    38</span></span>
<span><span class="co">##           biolink:gene_associated_with_condition    28</span></span>
<span><span class="co">##                                 biolink:model_of    27</span></span>
<span><span class="co">##                  biolink:has_mode_of_inheritance    26</span></span>
<span><span class="co">##              biolink:genetically_associated_with    11</span></span>
<span><span class="co">##                               biolink:related_to     8</span></span>
<span><span class="co">##    biolink:treats_or_applied_or_studied_to_treat     1</span></span></code></pre>
<p>The printout reports the number of nodes for each category, and the
number of edges for each predicate.</p>
<p>This information is also returned (invisibly) as a list; to suppress
the printed output we can add <code>quiet = TRUE</code>.</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">s</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/summary.html" class="external-link">summary</a></span><span class="op">(</span><span class="va">eds_marfan</span>, quiet <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste</a></span><span class="op">(</span><span class="st">"Total nodes:"</span>, <span class="va">s</span><span class="op">$</span><span class="va">total_nodes</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "Total nodes: 3000"</span></span></code></pre>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/base/paste.html" class="external-link">paste</a></span><span class="op">(</span><span class="st">"Total edges:"</span>, <span class="va">s</span><span class="op">$</span><span class="va">total_edges</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## [1] "Total edges: 7148"</span></span></code></pre>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">s</span><span class="op">$</span><span class="va">node_summary</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                             category count</span></span>
<span><span class="co">## 1                     biolink:Entity  3000</span></span>
<span><span class="co">## 2                 biolink:NamedThing  3000</span></span>
<span><span class="co">## 3           biolink:BiologicalEntity  2977</span></span>
<span><span class="co">## 4             biolink:ThingWithTaxon  2977</span></span>
<span><span class="co">## 5 biolink:DiseaseOrPhenotypicFeature  2846</span></span>
<span><span class="co">## 6          biolink:PhenotypicFeature  2736</span></span></code></pre>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/utils/head.html" class="external-link">head</a></span><span class="op">(</span><span class="va">s</span><span class="op">$</span><span class="va">edge_summary</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">##                                         predicate count</span></span>
<span><span class="co">## 1                             biolink:subclass_of  5244</span></span>
<span><span class="co">## 2                           biolink:has_phenotype  1709</span></span>
<span><span class="co">## 3                                  biolink:causes    56</span></span>
<span><span class="co">## 4 biolink:associated_with_increased_likelihood_of    38</span></span>
<span><span class="co">## 5          biolink:gene_associated_with_condition    28</span></span>
<span><span class="co">## 6                                biolink:model_of    27</span></span></code></pre>
<p>Finally, the returned list also includes <code>cats</code> and
<code>pred</code> entries, which are named lists containing all
available category and predicate labels for convenient auto-completion
in your favorite IDE.</p>
<p><br><br></p>
<p><img src="exploring_kgs_assets/autocomplete.png"></p>
<p><br><br></p>
<p>The resulting auto-completion inserts the appropriate backtics in
RStudio.</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">diseases</span> <span class="op">&lt;-</span> <span class="va">eds_marfan</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="../../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="st">"MONDO:0009159"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="../../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="va">s</span><span class="op">$</span><span class="va">preds</span><span class="op">$</span><span class="va">`biolink:has_phenotype`</span><span class="op">)</span></span>
<span></span>
<span><span class="va">diseases</span></span></code></pre></div>
<pre>Graph with 51 nodes and 50 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 51 of 51 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
type (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_gene
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Ehlers-Danlos syndrome, cardiac valvular type”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Ehlers-Danlos syndrome, cardiac valvular type is a form of
Ehlers-Danlos syndrome characterized by soft skin, skin
hyperextensibility, easy bruisability, atrophic scar formation, joint
hypermobility and cardiac valvular defects comprising mitral and/or
aortic valve insufficiency.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cardiac valvular form of Ehlers-Danlos syndrome”, “Cardiac valvular
form of autosomal recessive Ehlers-Danlos syndrome”, “Cardiac-valvular
EDS”, “Cardiac-valvular Ehlers-Danlos syndrome”, “EDS, cardiac valvular
type”, “EDSCV”, “Ehlers-Danlos syndrome, CARDIAC valvular type”,
“Ehlers-Danlos syndrome, arthrochalasis type”, “Ehlers-Danlos syndrome,
autosomal recessive, CARDIAC valvular form”, “Ehlers-Danlos syndrome,
autosomal recessive, Cardiac valvular form”, “cvEDS”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009159" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009159</a>”
</td>
<td style="text-align:left;">
c(“DOID:0080730”, “GARD:12613”, “MESH:C536200”, “OMIM:225320”,
“Orphanet:230851”, “SCTID:720858001”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hyperextensible skin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A condition in which the skin can be stretched beyond normal, and then
returns to its initial position.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Hyperelastic skin”, “Skin hyperelasticity”, “Skin
hyperextensibility”, “Stretchable skin”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000974" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000974</a>”
</td>
<td style="text-align:left;">
“UMLS:C0241074”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001382”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Joint hypermobility”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The ability of a joint to move beyond its normal range of motion.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Double-Jointed”, “Extensible joints”, “Flexible joints”,
“Hyperextensible joints”, “Increased joint mobility”, “Increased
mobility of joints”, “Joint hyperextensibility”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001382" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001382</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:298181000”, “UMLS:C1844820”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001653”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Mitral regurgitation”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormality of the mitral valve characterized by insufficiency or
incompetence of the mitral valve resulting in retrograde leaking of
blood through the mitral valve upon ventricular contraction.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Mitral incompetence”, “Mitral insufficiency”, “Mitral regurgitation,
mild”, “Mitral valve insufficiency”, “Mitral valve regurgitation”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001653" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001653</a>”
</td>
<td style="text-align:left;">
c(“Fyler:1151”, “MSH:D008944”, “SNOMEDCT_US:48724000”, “UMLS:C0026266”,
“UMLS:C3551535”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001654”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Abnormal heart valve morphology”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Any structural abnormality of a cardiac valve.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Abnormality of the heart valves”, “Valvular abnormality”, “Valvular
heart disease”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001654" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001654</a>”
</td>
<td style="text-align:left;">
c(“MSH:D006349”, “SNOMEDCT_US:368009”, “UMLS:C0018824”, “UMLS:C0241654”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000023”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Inguinal hernia”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Protrusion of the contents of the abdominal cavity through the inguinal
canal.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000023" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000023</a>”
</td>
<td style="text-align:left;">
c(“MEDDRA:10022016”, “MSH:D006552”, “SNOMEDCT_US:396232000”,
“UMLS:C0019294”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000486”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Strabismus”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A misalignment of the eyes so that the visual axes deviate from
bifoveal fixation. The classification of strabismus may be based on a
number of features including the relative position of the eyes, whether
the deviation is latent or manifest, intermittent or constant,
concomitant or otherwise and according to the age of onset and the
relevance of any associated refractive error.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cross-eyed”, “Squint”, “Squint eyes”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000486" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000486</a>”
</td>
<td style="text-align:left;">
c(“MSH:D013285”, “SNOMEDCT_US:128602000”, “SNOMEDCT_US:22066006”,
“UMLS:C0038379”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000508”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Ptosis”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The upper eyelid margin is positioned 3 mm or more lower than usual and
covers the superior portion of the iris (objective); or, the upper lid
margin obscures at least part of the pupil (subjective).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Blepharoptosis”, “Drooping upper eyelid”, “Eye drop”, “Eyelid
ptosis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000508" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000508</a>”
</td>
<td style="text-align:left;">
c(“MSH:D001763”, “SNOMEDCT_US:11934000”, “UMLS:C0005745”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000545”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Myopia”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormality of refraction characterized by the ability to see
objects nearby clearly, while objects in the distance appear blurry.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Close sighted”, “Near sighted”, “Near sightedness”,
“Nearsightedness”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000545" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000545</a>”
</td>
<td style="text-align:left;">
c(“MSH:D009216”, “SNOMEDCT_US:57190000”, “UMLS:C0027092”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000678”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Dental crowding”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Changes in alignment of teeth in the dental arch”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Crowded teeth”, “Dental crowding”, “Dental overcrowding”, “Inadequate
arch length for tooth size”, “Overcrowding of teeth”, “Tooth mass arch
size discrepancy”, “Tooth size discrepancy”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000678" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000678</a>”
</td>
<td style="text-align:left;">
c(“MSH:D008310”, “SNOMEDCT_US:12351004”, “SNOMEDCT_US:699222000”,
“UMLS:C0040433”, “UMLS:C1317785”, “UMLS:C4280617”, “UMLS:C4280618”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000767”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Pectus excavatum”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A defect of the chest wall characterized by a depression of the
sternum, giving the chest (""pectus"") a caved-in (""excavatum"")
appearance."”
</td>
<td style="text-align:left;min-width: 300px; ">
“Funnel chest”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000767" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000767</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:391987005”, “UMLS:C2051831”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000963”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Thin skin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Reduction in thickness of the skin, generally associated with a loss of
suppleness and elasticity of the skin.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Thin skin”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000963" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000963</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:277797007”, “UMLS:C0423757”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000978”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Bruising susceptibility”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An ecchymosis (bruise) refers to the skin discoloration caused by the
escape of blood into the tissues from ruptured blood vessels. This term
refers to an abnormally increased susceptibility to bruising. The
corresponding phenotypic abnormality is generally elicited on medical
history as a report of frequent ecchymoses or bruising without adequate
trauma.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Bruisability”, “Bruise easily”, “Bruising susceptibility”, “Easy
bruisability”, “Easy bruising”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000978" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000978</a>”
</td>
<td style="text-align:left;">
c(“MSH:D004438”, “SNOMEDCT_US:302227002”, “SNOMEDCT_US:424131007”,
“SNOMEDCT_US:425075004”, “SNOMEDCT_US:77643000”, “UMLS:C0013491”,
“UMLS:C0423798”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001027”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Soft, doughy skin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A skin texture that is unusually soft (and may feel silky), and has a
malleable consistency resembling that of dough.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Soft, doughy skin”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001027" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001027</a>”
</td>
<td style="text-align:left;">
“UMLS:C1849043”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001058”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Poor wound healing”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A reduced ability to heal cutaneous wounds.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Poor wound healing”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001058" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001058</a>”
</td>
<td style="text-align:left;">
“UMLS:C1851789”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001075”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Atrophic scars”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Scars that form a depression compared to the level of the surrounding
skin because of damage to the collagen, fat or other tissues below the
skin.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Sunken or indented skin due to damage”, “Thin, atrophic scars”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001075" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001075</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:239172000”, “SNOMEDCT_US:409766009”, “UMLS:C0162154”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001373”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Joint dislocation”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Displacement or malalignment of joints.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Joint dislocation”, “Joint dislocations”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001373" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001373</a>”
</td>
<td style="text-align:left;">
c(“MSH:D004204”, “SNOMEDCT_US:108367008”, “SNOMEDCT_US:87642003”,
“UMLS:C0012691”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001659”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Aortic regurgitation”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An insufficiency of the aortic valve, leading to regurgitation
(backward flow) of blood from the aorta into the left ventricle.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Aortic insufficiency”, “Aortic valve regurgitation”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001659" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001659</a>”
</td>
<td style="text-align:left;">
c(“MSH:D001022”, “SNOMEDCT_US:60234000”, “UMLS:C0003504”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001763”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Pes planus”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A foot where the longitudinal arch of the foot is in contact with the
ground or floor when the individual is standing; or, in a patient lying
supine, a foot where the arch is in contact with the surface of a flat
board pressed against the sole of the foot by the examiner with a
pressure similar to that expected from weight bearing; or, the height of
the arch is reduced.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Dropped arches”, “Fallen arches”, “Flat feet”, “Flat foot”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001763" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001763</a>”
</td>
<td style="text-align:left;">
c(“MSH:D005413”, “SNOMEDCT_US:203534009”, “SNOMEDCT_US:53226007”,
“UMLS:C0016202”, “UMLS:C0264133”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001822”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hallux valgus”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Lateral deviation of the great toe (i.e., in the direction of the
little toe).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Bunion”, “Lateral deviation of great toe”, “Lateral deviation of
halluces”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001822" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001822</a>”
</td>
<td style="text-align:left;">
c(“MSH:D000071378”, “MSH:D006215”, “SNOMEDCT_US:122480009”,
“SNOMEDCT_US:415692008”, “UMLS:C0006386”, “UMLS:C0018536”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002616”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Aortic root aneurysm”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormal localized widening (dilatation) of the aortic root.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Aortic root dilatation”, “Bulge in wall of root of large artery that
carries blood away from heart”, “Increased aortic root diameter”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002616" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002616</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:251036003”, “UMLS:C0238669”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002816”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Genu recurvatum”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormally increased extension of the knee joint, so that the knee
can bend backwards.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Back knee”, “Genu recurvata”, “Knee hyperextension”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002816" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002816</a>”
</td>
<td style="text-align:left;">
c(“MEDDRA:10018194”, “UMLS:C0546964”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002857”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Genu valgum”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The legs angle inward, such that the knees are close together and the
ankles far apart.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Genu valga”, “Genu valgus”, “Genua valga”, “Knee joint valgus
deformity”, “Knock knees”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002857" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002857</a>”
</td>
<td style="text-align:left;">
c(“MEDDRA:10023480”, “MSH:D056304”, “SNOMEDCT_US:299330008”,
“UMLS:C0576093”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0005180”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Tricuspid regurgitation”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Failure of the tricuspid valve to close sufficiently upon contraction
of the right ventricle, causing blood to regurgitate (flow backward)
into the right atrium.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Tricuspid insufficiency”, “Tricuspid valve regurgitation”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0005180" class="external-link uri">http://purl.obolibrary.org/obo/HP_0005180</a>”
</td>
<td style="text-align:left;">
c(“Fyler:1161”, “MSH:D014262”, “SNOMEDCT_US:111287006”, “UMLS:C0040961”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0006109”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Absent phalangeal crease”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Absence of one or more interphalangeal creases (i.e., of the transverse
lines in the skin between the phalanges of the fingers).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Absent interphalangeal creases”, “Aplasia of the interphalangeal
creases”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0006109" class="external-link uri">http://purl.obolibrary.org/obo/HP_0006109</a>”
</td>
<td style="text-align:left;">
c(“UMLS:C1862479”, “UMLS:C4020821”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0006201”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hypermobility of distal interphalangeal joints”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Increased mobility of outermost hinge joint”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0006201" class="external-link uri">http://purl.obolibrary.org/obo/HP_0006201</a>”
</td>
<td style="text-align:left;">
“UMLS:C1851811”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0100807”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Long fingers”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The middle finger is more than 2 SD above the mean for newborns 27 to
41 weeks EGA or above the 97th centile for children from birth to 16
years of age AND the five digits retain their normal length proportions
relative to each other (i.e., it is not the case that the middle finger
is the only lengthened digit), or, Fingers that appear
disproportionately long compared to the palm of the hand.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Long fingers”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0100807" class="external-link uri">http://purl.obolibrary.org/obo/HP_0100807</a>”
</td>
<td style="text-align:left;">
“UMLS:C1858091”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000218”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“High palate”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Height of the palate more than 2 SD above the mean (objective) or
palatal height at the level of the first permanent molar more than twice
the height of the teeth (subjective).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Elevated palate”, “High arched palate”, “High palate”, “High, arched
palate”, “High-arched palate”, “Increased palatal height”, “Ogival
palate”, “Palate high-arched”, “Palate, high-arched”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000218" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000218</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:27272007”, “UMLS:C0240635”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000414”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Bulbous nose”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Increased volume and globular shape of the anteroinferior aspect of the
nose.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Bulbous nasal tip”, “Bulbous nose”, “Potato nose”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000414" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000414</a>”
</td>
<td style="text-align:left;">
c(“MSH:C538354”, “UMLS:C0240543”, “UMLS:C1834118”, “UMLS:C1855751”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000574”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Thick eyebrow”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Increased density/number and/or increased diameter of eyebrow hairs.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Bushy eyebrows”, “Dense eyebrow”, “Heavy eyebrows”, “Hypertrichosis
of the eyebrow”, “Hypertrichosis of the eyebrows”, “Prominent eyebrows”,
“Thick eyebrow”, “Thick eyebrows”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000574" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000574</a>”
</td>
<td style="text-align:left;">
“UMLS:C1853487”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001250”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Seizure”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A seizure is an intermittent abnormality of nervous system physiology
characterised by a transient occurrence of signs and/or symptoms due to
abnormal excessive or synchronous neuronal activity in the brain.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Epilepsy”, “Epileptic seizure”, “Seizures”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001250" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001250</a>”
</td>
<td style="text-align:left;">
c(“MSH:D004827”, “MSH:D012640”, “SNOMEDCT_US:128613002”,
“SNOMEDCT_US:246545002”, “SNOMEDCT_US:313307000”,
“SNOMEDCT_US:84757009”, “SNOMEDCT_US:91175000”, “UMLS:C0014544”,
“UMLS:C0036572”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001263”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Global developmental delay”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A delay in the achievement of motor or mental milestones in the domains
of development of a child, including motor skills, speech and language,
cognitive skills, and social and emotional skills. This term should only
be used to describe children younger than five years of age.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cognitive delay”, “Delayed cognitive development”, “Delayed
development”, “Delayed developmental milestones”, “Delayed intellectual
development”, “Delayed milestones”, “Delayed psychomotor development”,
“Developmental delay”, “Developmental delay in early childhood”,
“Developmental delay, global”, “Developmental retardation”, “Lack of
psychomotor development”, “Mental and motor retardation”, “Motor and
developmental delay”, “Motormental retardation”, “Psychomotor delay”,
“Psychomotor development deficiency”, “Psychomotor development failure”,
“Psychomotor developmental delay”, “Retarded development”, “Retarded
mental development”, “Retarded psychomotor development”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001263" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001263</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:224958001”, “UMLS:C0557874”, “UMLS:C1864897”,
“UMLS:C4020875”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001519”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Disproportionate tall stature”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A tall and slim body build with increased arm span to height ratio
(&gt;1.05) and a reduced upper-to-lower segment ratio (&lt;0.85), i.e.,
unusually long arms and legs. The extremities as well as the hands and
feet are unusually slim.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Dolichostenomelia”, “Marfanoid body habitus”, “Marfanoid habitus”,
“Reduced upper-lower segment ratio”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001519" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001519</a>”
</td>
<td style="text-align:left;">
c(“MSH:D054119”, “SNOMEDCT_US:62250003”, “UMLS:C0003706”,
“UMLS:C1836996”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001631”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Atrial septal defect”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Atrial septal defect (ASD) is a congenital abnormality of the
interatrial septum that enables blood flow between the left and right
atria via the interatrial septum.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“ASD”, “An opening in the wall separating the top two chambers of the
heart”, “Atria septal defect”, “Atrial septum defect”, “Atrioseptal
defect”, “Defect in the atrial septum”, “Hole in heart wall separating
two upper heart chambers”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001631" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001631</a>”
</td>
<td style="text-align:left;">
c(“Fyler:2050”, “ICD-10:Q21.1”, “MSH:D006344”, “SNOMEDCT_US:253366007”,
“SNOMEDCT_US:405752007”, “SNOMEDCT_US:70142008”, “UMLS:C0018817”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001634”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Mitral valve prolapse”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“One or both of the leaflets (cusps) of the mitral valve bulges back
into the left atrium upon contraction of the left ventricle.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001634" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001634</a>”
</td>
<td style="text-align:left;">
c(“Fyler:1533”, “MSH:D008945”, “SNOMEDCT_US:409712001”,
“SNOMEDCT_US:8074002”, “UMLS:C0026267”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001712”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Left ventricular hypertrophy”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Enlargement or increased size of the heart left ventricle.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Heart left ventricle hypertrophy”, “Left ventricular wall
hypertrophy”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001712" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001712</a>”
</td>
<td style="text-align:left;">
c(“Fyler:3608”, “MSH:D017379”, “SNOMEDCT_US:55827005”, “UMLS:C0149721”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001848”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Calcaneovalgus deformity”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“This is a postural deformity in which the foot is positioned up against
the tibia. The heel (calcaneus) is positioned downward (that is, the
ankle is flexed upward), and the heel is turned outward (valgus).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Calcaneovalgus”, “Calcaneovalgus Foot”, “Foot and ankle bend up
toward shin of leg”, “Valgus position of the calcaneus”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001848" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001848</a>”
</td>
<td style="text-align:left;">
“UMLS:C1860450”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001852”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Sandal gap”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A widely spaced gap between the first toe (the great toe) and the
second toe.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Gap between 1st and 2nd toes”, “Gap between first and second toe”,
“Increased space between first and second toes”, “Sandal gap between
first and second toes”, “Space between great toe and second toe”, “Wide
space between 1st, 2nd toes”, “Wide space between first and second
toes”, “Wide-spaced big toe”, “Widely spaced 1st-2nd toes”, “Widely
spaced first and second toes”, “Widened gap 1st-2nd toes”, “Widened gap
first and second toe”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001852" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001852</a>”
</td>
<td style="text-align:left;">
“UMLS:C1840069”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002094”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Dyspnea”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Difficult or labored breathing. Dyspnea is a subjective feeling only
the patient can rate, e.g., on a Borg scale.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Abnormal breathing”, “Breathing difficulty”, “Difficult to breathe”,
“Difficulty breathing”, “Dyspnoea”, “Panting”, “Shortness of breath”,
“Trouble breathing”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002094" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002094</a>”
</td>
<td style="text-align:left;">
c(“MSH:D004417”, “SNOMEDCT_US:230145002”, “SNOMEDCT_US:267036007”,
“UMLS:C0013404”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002342”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Intellectual disability, moderate”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Moderate mental retardation is defined as an intelligence quotient (IQ)
in the range of 35-49.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“IQ between 34 and 49”, “Mental retardation, moderate”, “Moderate
mental deficiency”, “Moderate mental retardation”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002342" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002342</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:61152003”, “UMLS:C0026351”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002751”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Kyphoscoliosis”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormal curvature of the spine in both a coronal (lateral) and
sagittal (back-to-front) plane.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002751" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002751</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:405773007”, “UMLS:C0575158”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002944”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Thoracolumbar scoliosis”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Scoliosis, thoracolumbar”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002944" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002944</a>”
</td>
<td style="text-align:left;">
“UMLS:C0749379”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0004322”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Short stature”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A height below that which is expected according to age and gender
norms. Although there is no universally accepted definition of short
stature, many refer to ""short stature"" as height more than 2 standard
deviations below the mean for age and gender (or below the 3rd
percentile for age and gender dependent norms)."”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Decreased body height”, “Height less than 3rd percentile”, “Short
stature”, “Small stature”, “Stature below 3rd percentile”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0004322" class="external-link uri">http://purl.obolibrary.org/obo/HP_0004322</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:237836003”, “UMLS:C0349588”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0010444”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Pulmonary insufficiency”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The retrograde (backwards) flow of blood through the pulmonary valve
into the right ventricle during diastole.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Pulmonary incompetence”, “Pulmonary valve regurgitation”, “Pulmonic
regurgitation”, “Puolmonary valve insufficiency”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0010444" class="external-link uri">http://purl.obolibrary.org/obo/HP_0010444</a>”
</td>
<td style="text-align:left;">
c(“MSH:D011665”, “SNOMEDCT_US:91434003”, “UMLS:C0034088”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0012378”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Fatigue”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A subjective feeling of tiredness characterized by a lack of energy and
motivation.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Fatigue”, “Tired”, “Tiredness”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0012378" class="external-link uri">http://purl.obolibrary.org/obo/HP_0012378</a>”
</td>
<td style="text-align:left;">
c(“MSH:D005221”, “SNOMEDCT_US:248274002”, “SNOMEDCT_US:84229001”,
“UMLS:C0015672”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0012717”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Severe conductive hearing impairment”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A severe form of conductive hearing impairment.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Conductive hearing loss, severe”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0012717" class="external-link uri">http://purl.obolibrary.org/obo/HP_0012717</a>”
</td>
<td style="text-align:left;">
“UMLS:C4021074”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0031610”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Recurrent shoulder dislocation”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Shoulder dislocation occurring repeated times.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Multiple shoulder dislocation”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0031610" class="external-link uri">http://purl.obolibrary.org/obo/HP_0031610</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0032523”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Tendon thickening”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An abnormal increase in the thickness (diameter) of a tendon.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0032523" class="external-link uri">http://purl.obolibrary.org/obo/HP_0032523</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0100550”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Tendon rupture”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Breakage (tear) of a tendon.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Rupture of tendons”, “Ruptured tendon”, “Tendon rupture”,
“Tendon/muscle rupture”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0100550" class="external-link uri">http://purl.obolibrary.org/obo/HP_0100550</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:415749005”, “UMLS:C0151937”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0500041”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Myopic astigmatism”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A condition where one or both of the two principal meridians focus in
the front of the retina when the eye is at rest.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0500041" class="external-link uri">http://purl.obolibrary.org/obo/HP_0500041</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000977”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Soft skin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Subjective impression of increased softness upon palpation of the
skin.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Soft skin”, “Velvety skin”, “Velvety skin texture”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000977" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000977</a>”
</td>
<td style="text-align:left;">
c(“UMLS:C0241178”, “UMLS:C1844592”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 50 of 50 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
from
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
to
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
predicate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
primary_knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
agent_type
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_level
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
aggregator_knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
frequency_qualifier
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_evidence
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_total
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_quotient
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_count
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_percentage
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
onset_qualifier
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
publications
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
qualifiers
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_predicate
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583938-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040281”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001382”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:2258392d-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040281”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001653”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583935-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040281”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001654”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62cf8-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040281”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000023”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:2258392c-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000486”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62cfa-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000508”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62cfb-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000545”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62cfc-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000678”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62cfd-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000767”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583931-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000963”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583937-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000978”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583932-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001027”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d01-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001058”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d02-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001075”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:2258392e-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001373”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d04-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001659”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583934-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
19
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001763”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583936-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
20
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001822”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d07-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
21
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002616”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d08-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
22
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002816”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583930-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
23
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002857”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0a-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
24
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0005180”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0b-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
25
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0006109”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0c-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
26
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0006201”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0d-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
27
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0100807”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0e-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040282”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
28
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000218”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d0f-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
29
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000414”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d10-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
30
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000574”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d11-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
31
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001250”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d12-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
32
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001263”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d13-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
33
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001519”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d14-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
34
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001631”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d15-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
35
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001634”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583939-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
36
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001712”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d17-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
37
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001848”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:2258392f-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
38
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0001852”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d19-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
39
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002094”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1a-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
40
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002342”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1b-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
41
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002751”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1c-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
42
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0002944”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1d-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
43
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0004322”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1e-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
44
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0010444”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d1f-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
45
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0012378”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d20-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
46
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0012717”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d21-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
47
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0031610”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d22-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
48
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0032523”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d23-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
49
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0100550”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d24-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
50
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0500041”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:27f62d25-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Orphanet:230851”
</td>
<td style="text-align:left;">
“HP:0040283”
</td>
<td style="text-align:left;">
“ECO:0000304”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
51
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000977”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583933-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
</div>
</details><p><br></p>
</div>
<div class="section level2">
<h2 id="engine-example_graph">Engine <code>example_graph()</code><a class="anchor" aria-label="anchor" href="#engine-example_graph"></a>
</h2>
<p>Fetching a sample of data from a KG is another convenient way to
explore its contents, but a random sample is unlikely to illustrate the
diversity of available node categories, edge predicates, and information
associated with nodes and edges of different types.</p>
<p>To serve this need <code>monarchr</code> provides an
<code><a href="../../reference/example_graph.html">example_graph()</a></code> function, which fetches a sample of nodes
and edges that are guaranteed to represent every available category and
every available predicate. When using this method, it is important to
remember that nodes frequently belong to multiple categories, and the
<code>pcategory</code> (“primary category”) column represents one of a
set of chosen categories to represent the node. The choice of category
shown in <code>pcategory</code> is defined by <code>monarchr</code>, not
the KG itself, and is configurable.</p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">ex_g</span> <span class="op">&lt;-</span> <span class="va">eds_marfan</span> <span class="op">|&gt;</span> <span class="fu"><a href="../../reference/example_graph.html">example_graph</a></span><span class="op">(</span><span class="op">)</span></span>
<span><span class="va">ex_g</span></span></code></pre></div>
<pre>Graph with 12 nodes and 10 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 12 of 12 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
type (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_gene
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Marfan syndrome”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A disorder of the connective tissue. Connective tissue provides
strength and flexibility to structures throughout the body such as
bones, ligaments, muscles, walls of blood vessels, and heart valves.
Marfan syndrome affects most organs and tissues, especially the
skeleton, lungs, eyes, heart, and the large blood vessel that
distributes blood from the heart to the rest of the body (the aorta). It
is caused by mutations in the FBN1 gene, which provides instructions for
making a protein called fibrillin-1. Marfan syndrome is inherited in an
autosomal dominant pattern. At least 25% of cases are due to a new (de
novo) mutation. Treatment is based on the signs and symptoms in each
person.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“MFS”, “MFS1”, “Marfan syndrome”, “Marfan syndrome type 1”, “Marfan
syndrome, type 1”, “Marfan’s syndrome”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0007947" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0007947</a>”
</td>
<td style="text-align:left;">
c(“DOID:14323”, “GARD:16535”, “ICD9:759.82”, “MESH:D008382”,
“MedDRA:10026829”, “NANDO:1200644”, “NANDO:2200968”, “NCIT:C34807”,
“NORD:1403”, “OMIM:154700”, “Orphanet:284963”, “Orphanet:558”,
“SCTID:19346006”, “UMLS:C0024796”, “icd11.foundation:236564145”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0020066”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Ehlers-Danlos syndrome”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“The Ehlers–Danlos syndromes (EDS) are a clinically and genetically
heterogeneous group of heritable connective tissue disorders (HCTDs)
characterized by joint hypermobility, skin hyperextensibility, and
tissue fragility.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Danlos Disease, Ehlers”, “Danlos disease”, “Disease, Ehlers Danlos”,
“Disease, Ehlers-Danlos”, “Dystrophia mesodermalis congenita”, “ED
syndrome”, “EDS”, “Ehler Danlos Syndrome”, “Ehlers Danlos Disease”,
“Ehlers Danlos Syndrome”, “Ehlers Danlos syndrome”, “Ehlers-Danlos
Disease”, “Ehlers-Danlos syndromes”, “Fibrodysplasia elastica
generalisata”, “Hereditary collagen dysplasia”, “Meekeren-Ehlers-Danlos
syndrome”, “Syndrome, Ehlers-Danlos”, “danlos ehlers syndrome”, “elastic
skin”, “skin elastic”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0020066" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0020066</a>”
</td>
<td style="text-align:left;">
c(“DOID:13359”, “GARD:6322”, “ICD10CM:Q79.6”, “ICD9:756.83”,
“MESH:D004535”, “MedDRA:10014316”, “NANDO:1200645”, “NANDO:2200607”,
“NCIT:C34568”, “NORD:1080”, “OMIMPS:130000”, “Orphanet:98249”,
“SCTID:398114001”, “UMLS:C0013720”, “icd11.foundation:1122707206”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Ehlers-Danlos syndrome, cardiac valvular type”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Ehlers-Danlos syndrome, cardiac valvular type is a form of
Ehlers-Danlos syndrome characterized by soft skin, skin
hyperextensibility, easy bruisability, atrophic scar formation, joint
hypermobility and cardiac valvular defects comprising mitral and/or
aortic valve insufficiency.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cardiac valvular form of Ehlers-Danlos syndrome”, “Cardiac valvular
form of autosomal recessive Ehlers-Danlos syndrome”, “Cardiac-valvular
EDS”, “Cardiac-valvular Ehlers-Danlos syndrome”, “EDS, cardiac valvular
type”, “EDSCV”, “Ehlers-Danlos syndrome, CARDIAC valvular type”,
“Ehlers-Danlos syndrome, arthrochalasis type”, “Ehlers-Danlos syndrome,
autosomal recessive, CARDIAC valvular form”, “Ehlers-Danlos syndrome,
autosomal recessive, Cardiac valvular form”, “cvEDS”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009159" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009159</a>”
</td>
<td style="text-align:left;">
c(“DOID:0080730”, “GARD:12613”, “MESH:C536200”, “OMIM:225320”,
“Orphanet:230851”, “SCTID:720858001”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016002”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Ehlers-Danlos syndrome, kyphoscoliotic type 1”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A form of Ehlers-Danlos syndrome characterized by severe hypotonia and
kyphoscoliosis at birth, generalized joint hyperextensibility and ocular
globe fragility.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“EDS 6”, “EDS 6 (formerly)”, “EDS VI”, “EDS VIA”, “EDS, kyphoscoliotic
type”, “EDS, oculoscoliotic type”, “EDS6”, “EDS6A, formerly”,
“EDSKSCL1”, “Ehlers-Danlos syndrome kyphoscoliotic type”, “Ehlers-Danlos
syndrome oculoscoliotic type”, “Ehlers-Danlos syndrome type 6
(formerly)”, “Ehlers-Danlos syndrome type 6A”, “Ehlers-Danlos syndrome
type 6A (formerly)”, “Ehlers-Danlos syndrome, kyphoscoliosis type”,
“Ehlers-Danlos syndrome, kyphoscoliotic type”, “Ehlers-Danlos syndrome,
kyphoscoliotic type 1”, “Ehlers-Danlos syndrome, kyphoscoliotic type,
1”, “Ehlers-Danlos syndrome, ocular-scoliotic type”, “Ehlers-Danlos
syndrome, oculoscoliotic type”, “Ehlers-Danlos syndrome, type 6”,
“Ehlers-Danlos syndrome, type 6 A”, “Ehlers-Danlos syndrome, type VI”,
“Ehlers-Danlos syndrome, type VIA”, “Ehlers-Danlos syndrome, type VIA,
formerly”, “Ehlers-Danlos syndrome, type Via”, “Ehlers-Danlos syndrome,
type Via, formerly”, “kEDS”, “kyphoscoliotic EDS”, “kyphoscoliotic
Ehlers-Danlos syndrome”, “kyphoscoliotic Ehlers-Danlos syndrome due to
lysyl hydroxylase 1 deficiency”, “nevo syndrome”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016002" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016002</a>”
</td>
<td style="text-align:left;">
c(“DOID:0080734”, “GARD:22216”, “MESH:C536198”, “NANDO:1200649”,
“NANDO:2201259”, “NCIT:C125700”, “OMIM:225400”, “Orphanet:1900”,
“SCTID:718211004”, “UMLS:C0268342”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“ZFIN:ZDB-FISH-220830-1”
</td>
<td style="text-align:left;">
“biolink:Genotype”
</td>
<td style="text-align:left;">
“b3galt6<sup>cmg20/cmg20</sup>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Danio rerio”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:Genotype”, “biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“ZFIN”
</td>
<td style="text-align:left;">
“alliance_genotype_nodes”
</td>
<td style="text-align:left;">
“NCBITaxon:7955”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:3603”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“FBN1”
</td>
<td style="text-align:left;">
“FBN1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
c(“FBN”, “MASS”, “MFS1”, “Marfan syndrome”, “OCTD”, “SGS”, “WMS”,
“asprosin”, “fibrillin 1 (Marfan syndrome)”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000166147”, “OMIM:134797”)
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“fibrillin 1”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hyperextensible skin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A condition in which the skin can be stretched beyond normal, and then
returns to its initial position.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Hyperelastic skin”, “Skin hyperelasticity”, “Skin
hyperextensibility”, “Stretchable skin”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000974" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000974</a>”
</td>
<td style="text-align:left;">
“UMLS:C0241074”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000007”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Autosomal recessive inheritance”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A mode of inheritance that is observed for traits related to a gene
encoded on one of the autosomes (i.e., the human chromosomes 1-22) in
which a trait manifests in individuals with two pathogenic alleles,
either homozygotes (two copies of the same mutant allele) or compound
heterozygotes (whereby each copy of a gene has a distinct mutant
allele).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Autosomal recessive”, “Autosomal recessive form”, “Autosomal
recessive predisposition”, “biallelic_autosomal”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000007" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000007</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:258211005”, “UMLS:C0441748”, “UMLS:C4020899”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0021136”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“rare”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A disease or disorder is defined as rare in Europe when it affects
fewer than 1 in 2000. A disease or disorder is defined as rare in the
USA when it affects fewer than 200,000 persons at any given time. Here
we take the European definition to be consistent with Orphanet.”
</td>
<td style="text-align:left;min-width: 300px; ">
“rare (European definition)”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0021136" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0021136</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“CHEBI:50845”
</td>
<td style="text-align:left;">
“biolink:MolecularEntity”
</td>
<td style="text-align:left;">
“doxycycline”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Tetracycline in which the 5beta-hydrogen is replaced by a hydroxy
group, while the 6alpha-hydroxy group is replaced by hydrogen. A
semi-synthetic tetracycline antibiotic, it is used to inhibit bacterial
protein synthesis and treat non-gonococcal urethritis and cervicitis,
exacerbations of bronchitis in patients with chronic obstructive
pulmonary disease (COPD), and adult periodontitis.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“(4S,4AR,5S,5AR,6R,12AS)-4-(DIMETHYLAMINO)-3,5,10,12,12A-PENTAHYDROXY-6-METHYL-1,11-DIOXO-1,4,4A,5,5A,6,11,12A-OCTAHYDROTETRACENE-2-CARBOXAMIDE”,
“(4S,4aR,5S,5aR,6R,12aS)-4-(dimethylamino)-3,5,10,12,12a-pentahydroxy-6-methyl-1,11-dioxo-1,4,4a,5,5a,6,11,12a-octahydrotetracene-2-carboxamide”,
“5-hydroxy-alpha-6-deoxytetracycline”, “6alpha-deoxy-5-oxytetracycline”,
“Doxycyclin”, “Doxycycline”, “Jenacyclin”, “Supracyclin”, “Vibramycin”,
“doxiciclina”, “doxycycline”, “doxycycline (anhydrous)”, “doxycyclinum”)
</td>
<td style="text-align:left;">
c(“biolink:ChemicalEntity”, “biolink:ChemicalEntityOrGeneOrGeneProduct”,
“biolink:ChemicalEntityOrProteinOrPolypeptide”,
“biolink:ChemicalOrDrugOrTreatment”, “biolink:Entity”,
“biolink:MolecularEntity”, “biolink:NamedThing”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/CHEBI_50845" class="external-link uri">http://purl.obolibrary.org/obo/CHEBI_50845</a>”
</td>
<td style="text-align:left;">
c(“Beilstein:3041790”, “CAS:564-25-0”, “Chemspider:10469369”,
“DrugBank:DB00254”, “Drug_Central:961”, “HMDB:HMDB0014399”,
“KEGG:C06973”, “KEGG:D07876”, “KNApSAcK:C00017127”,
“LIPID_MAPS_instance:LMPK07000001”, “MetaCyc:CPD-19256”, “PDBeChem:DXT”,
“PMCID:PMC8464303”, “PMCID:PMC8466018”, “<a href="PMID:11477376" class="uri">PMID:11477376</a>”, “<a href="PMID:11850258" class="uri">PMID:11850258</a>”, “<a href="PMID:11891205" class="uri">PMID:11891205</a>”, “<a href="PMID:1650428" class="uri">PMID:1650428</a>”, “<a href="PMID:16651635" class="uri">PMID:16651635</a>”, “<a href="PMID:19630297" class="uri">PMID:19630297</a>”, “<a href="PMID:23698443" class="uri">PMID:23698443</a>”, “<a href="PMID:24134630" class="uri">PMID:24134630</a>”, “<a href="PMID:34128838" class="uri">PMID:34128838</a>”, “<a href="PMID:34228348" class="uri">PMID:34228348</a>”, “<a href="PMID:34318509" class="uri">PMID:34318509</a>”, “<a href="PMID:34338231" class="uri">PMID:34338231</a>”, “<a href="PMID:34355683" class="uri">PMID:34355683</a>”, “<a href="PMID:34551605" class="uri">PMID:34551605</a>”, “<a href="PMID:34577552" class="uri">PMID:34577552</a>”, “Patent:US3019260”, “Patent:US3200149”,
“Reaxys:3041790”, “VSDB:1840”, “Wikipedia:Doxycycline”)
</td>
<td style="text-align:left;">
“CHEBI”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“CLINVAR:200198”
</td>
<td style="text-align:left;">
“biolink:SequenceVariant”
</td>
<td style="text-align:left;">
“NM_000138.5(FBN1):c.7577A&gt;G (p.Asn2526Ser)”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“CA017256”
</td>
<td style="text-align:left;">
“CLINVAR”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“HGNC:3603”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CLINVAR:200041”
</td>
<td style="text-align:left;">
“biolink:SequenceVariant”
</td>
<td style="text-align:left;">
“NM_000138.5(FBN1):c.4214T&gt;G (p.Leu1405Arg)”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“CA014858”
</td>
<td style="text-align:left;">
“CLINVAR”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“HGNC:3603”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 10 of 10 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
from
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
to
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
predicate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
primary_knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
agent_type
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_level
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
aggregator_knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
frequency_qualifier
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_evidence
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_total
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_quotient
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_count
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_percentage
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
onset_qualifier
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
publications
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
qualifiers
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_predicate
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“CLINVAR:200198”
</td>
<td style="text-align:left;">
“biolink:associated_with_increased_likelihood_of”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“infores:clingen”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“clingen_variant_edges”
</td>
<td style="text-align:left;">
“1ca9ddde-e045-4cf4-81f8-91dc85e3844b”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:Entity|biolink:EntityToDiseaseAssociationMixin|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:FrequencyQualifierMixin|biolink:VariantToDiseaseAssociation|biolink:VariantToEntityAssociationMixin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“reviewed_by_expert_panel”
</td>
<td style="text-align:left;">
“Likely Pathogenic”
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“HGNC:3603”
</td>
<td style="text-align:left;">
“biolink:causes”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“infores:omim”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:medgen|infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:6307821c-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:CausalGeneToDiseaseAssociation|biolink:Entity|biolink:EntityToDiseaseAssociationMixin|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:GeneToDiseaseAssociation|biolink:GeneToDiseaseOrPhenotypicFeatureAssociation|biolink:GeneToEntityAssociationMixin|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
“OMIM:154700”
</td>
<td style="text-align:left;">
“NCBIGene:2200”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“HGNC:3603”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:63670796-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:CorrelatedGeneToDiseaseAssociation|biolink:Entity|biolink:EntityToDiseaseAssociationMixin|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:GeneToDiseaseAssociation|biolink:GeneToDiseaseOrPhenotypicFeatureAssociation|biolink:GeneToEntityAssociationMixin|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
“Orphanet:284963”
</td>
<td style="text-align:left;">
“NCBIGene:2200”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“CLINVAR:200041”
</td>
<td style="text-align:left;">
“biolink:genetically_associated_with”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“infores:clingen”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“clingen_variant_edges”
</td>
<td style="text-align:left;">
“04c2ed0b-cf3b-418a-a019-c96d62eca315”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:Entity|biolink:EntityToDiseaseAssociationMixin|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:FrequencyQualifierMixin|biolink:VariantToDiseaseAssociation|biolink:VariantToEntityAssociationMixin”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“Uncertain Significance”
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_mode_of_inheritance”
</td>
<td style="text-align:left;">
“HP:0000007”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_mode_of_inheritance_edges”
</td>
<td style="text-align:left;">
“uuid:677a63c5-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseOrPhenotypicFeatureToEntityAssociationMixin|biolink:DiseaseOrPhenotypicFeatureToGeneticInheritanceAssociation|biolink:Entity”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:has_phenotype”
</td>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“infores:hpo-annotations”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“hpoa_disease_to_phenotype_edges”
</td>
<td style="text-align:left;">
“uuid:22583938-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:DiseaseToEntityAssociationMixin|biolink:DiseaseToPhenotypicFeatureAssociation|biolink:Entity|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:EntityToPhenotypicFeatureAssociationMixin|biolink:FrequencyQualifierMixin|biolink:FrequencyQuantifier|biolink:RelationshipQuantifier”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“OMIM:225320”
</td>
<td style="text-align:left;">
“HP:0040281”
</td>
<td style="text-align:left;">
“ECO:0000501”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
“ZFIN:ZDB-FISH-220830-1”
</td>
<td style="text-align:left;">
“biolink:model_of”
</td>
<td style="text-align:left;">
“MONDO:0020066”
</td>
<td style="text-align:left;">
“infores:zfin”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:agrkb|infores:monarchinitiative”
</td>
<td style="text-align:left;">
“alliance_disease_edges”
</td>
<td style="text-align:left;">
“97264874-3a20-11ef-b982-c3fd19cce226”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:Entity|biolink:EntityToDiseaseAssociationMixin|biolink:EntityToFeatureOrDiseaseQualifiersMixin|biolink:FrequencyQualifierMixin|biolink:GenotypeToDiseaseAssociation|biolink:GenotypeToEntityAssociationMixin”
</td>
<td style="text-align:left;">
“DOID:13359”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
“MONDO:0016002”
</td>
<td style="text-align:left;">
“biolink:related_to”
</td>
<td style="text-align:left;">
“MONDO:0021136”
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative|infores:phenio”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:00c9aab5-db6e-4c75-9aa7-2a175e48d9a4" class="uri">urn:uuid:00c9aab5-db6e-4c75-9aa7-2a175e48d9a4</a>”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:Entity”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
“MONDO:0009159”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0020066”
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative|infores:phenio”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:0a801142-530b-49bd-87c5-77f082e64c30" class="uri">urn:uuid:0a801142-530b-49bd-87c5-77f082e64c30</a>”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:Entity”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“CHEBI:50845”
</td>
<td style="text-align:left;">
“biolink:treats_or_applied_or_studied_to_treat”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“infores:ctd”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“ctd_chemical_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:234944a0-4007-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
“biolink:Association|biolink:ChemicalEntityToEntityAssociationMixin|biolink:ChemicalToDiseaseOrPhenotypicFeatureAssociation|biolink:ChemicalToEntityAssociationMixin|biolink:Entity|biolink:EntityToDiseaseOrPhenotypicFeatureAssociationMixin”
</td>
<td style="text-align:left;">
“MESH:D008382”
</td>
<td style="text-align:left;">
“MESH:D004318”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“<a href="PMID:18178469" class="uri">PMID:18178469</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
</tbody>
</table>
</div>
</details><p><br></p>
<p>Note that this method makes no other guarantees: the sample is not
random, the resulting graph may not be connected, the result is not the
smallest possible graph that contains all categories and predicates, and
nodes and edges may not contain complete data for all possible
properties for their respective types. Still, browsing the resulting
graph in tabular form as above can quickly reveal the bulk of
information available in a KG for further targeted exploration with
<code><a href="../../reference/fetch_nodes.html">fetch_nodes()</a></code> and <code><a href="../../reference/expand.html">expand()</a></code>.</p>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Shawn O’Neil, Brian Schilder.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.3.</p>
</div>

      </footer>
</div>






  </body>
</html>
