<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>MonarchR Basics • monarchr</title>
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.7.1/jquery.min.js" integrity="sha512-v2CJ7UaYy4JwqLDIrZUI/4hqeoQieOmAZNXBeQyjo21dadnwR+8ZaIJVT8EE2iyI61OV8e6M8PP2/4hpQINQ/g==" crossorigin="anonymous" referrerpolicy="no-referrer"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="../bootstrap-toc.css">
<script src="../bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="../pkgdown.css" rel="stylesheet">
<script src="../pkgdown.js"></script><link href="../extra.css" rel="stylesheet">
<meta property="og:title" content="MonarchR Basics">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">


    <div class="container template-article">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="../index.html">monarchr</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="">1.1.0</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="../articles/monarchr.html">Get started</a>
</li>
<li>
  <a href="../reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" data-bs-toggle="dropdown" aria-expanded="false">
    Articles

    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li class="divider">
    </li>
<li class="dropdown-header">Examples</li>
    <li>
      <a href="../articles/examples/alzheimers_phenotypes.html">Categorizing Alzheimers Phenotypes</a>
    </li>
  </ul>
</li>
<li>
  <a href="../news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/monarch-initiative/monarchr/" class="external-link">
    <span class="fab fa-github fa-lg"></span>

  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->



      </header><script src="monarchr_files/kePrint-0.0.1/kePrint.js"></script><link href="monarchr_files/lightable-0.0.1/lightable.css" rel="stylesheet">
<div class="row">
  <div class="col-md-9 contents">
    <div class="page-header toc-ignore">
      <h1 data-toc-skip>MonarchR Basics</h1>
                        <h4 data-toc-skip class="author">Shawn T
O’Neil</h4>
            
            <h4 data-toc-skip class="date">Vignette updated:
<i>Aug-05-2024</i>
</h4>
      
      <small class="dont-index">Source: <a href="https://github.com/monarch-initiative/monarchr/blob/expand_dup_ids/vignettes/monarchr.Rmd" class="external-link"><code>vignettes/monarchr.Rmd</code></a></small>
      <div class="hidden name"><code>monarchr.Rmd</code></div>

    </div>

    
    
<div class="section level2">
<h2 id="background-biomedical-knowledge-graphs">Background: Biomedical Knowledge Graphs<a class="anchor" aria-label="anchor" href="#background-biomedical-knowledge-graphs"></a>
</h2>
<p>Knowledge graphs (KGs) represent entities (such as genes, diseases,
or phenotypes) and the relationships between them; for example that the
<code>CFTR</code> gene <code>causes</code> the disease
<code>Cystic fibrosis</code>. In the <a href="https://monarchiniative.org" class="external-link">Monarch Initiative</a> KG, this
information is stored in a <em>directed, labeled property graph</em>,
with properties attached to the entities (nodes) and relationships
(edges). The <a href="https://github.com/biolink/kgx/blob/master/specification/kgx-format.md" class="external-link">KGX
format</a> used by Monarch provides a simple specification for KGs:</p>
<ul>
<li>Nodes must have an <code>id</code> property</li>
<li>Nodes must have a <code>category</code> property, as a multi-valued
set of category labels</li>
<li>Edges must have <code>subject</code>, <code>predicate</code>, and
<code>object</code> properties, representing the source, relationship,
and destination of the edge.</li>
<li>Nodes and edges may have additional properties, and these may be
multi-valued.</li>
</ul>
<p>In the following figure, these required properties are shown in
bold:</p>
<center>
<img src="assets/cftr_cf_edge.png" style="width:90.0%">
</center>
<p>The <code>monarchr</code> package provides access to the cloud-hosted
Monarch Initiative KG, and <a href="https://kghub.org" class="external-link">others in KGX
format</a> via graph-database and file-based “engines”, with a simple
but flexible set of query functions. The example below finds all of the
sub-types of <a href="https://monarchinitiative.org/MONDO:0001982" class="external-link">Neimann-Pick
Disease</a>, and all of the genes associated with those. (You an use the
<a href="https://monarchiniative.org" class="external-link">Monarch Initiative</a> website to
identify IDs for genes, diseases, phenotypes and more, or
<code>search_nodes()</code> described below.)</p>
<div class="sourceCode" id="cb1"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># MONDO:0001982 Niemann-Pick Disease (13 subtypes)</span></span>
<span></span>
<span><span class="va">g</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/monarch_engine.html">monarch_engine</a></span><span class="op">(</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MONDO:0001982"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="st">"biolink:subclass_of"</span>, direction <span class="op">=</span> <span class="st">"in"</span>, transitive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>categories <span class="op">=</span> <span class="st">"biolink:Gene"</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Fetching; counting matching nodes...  total: 1.</span></span>
<span><span class="co">## Fetching; fetched 1 of 1</span></span>
<span><span class="co">## Expanding; counting matching edges...  total: 13.</span></span>
<span><span class="co">## Expanding; fetched 13 of 13 edges.</span></span>
<span><span class="co">## Expanding; counting matching edges...  total: 16.</span></span>
<span><span class="co">## Expanding; fetched 16 of 16 edges.</span></span></code></pre>
<div class="sourceCode" id="cb3"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">g</span><span class="op">)</span></span></code></pre></div>
<pre><code><span><span class="co">## Using <span style="color: #0000BB;">"sugiyama"</span> as default layout</span></span></code></pre>
<div class="sourceCode" id="cb5"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">g</span></span></code></pre></div>
<p><img src="monarchr_files/figure-html/unnamed-chunk-2-1.png" width="960"></p>
<pre>Graph with 17 nodes and 29 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 17 of 17 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0001982”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“A group of inherited, severe metabolic disorders in which sphingomyelin
accumulates in lysosomes in cells. The lysosomes normally transport
material through and out of the cell.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Niemann-Pick disease with cholesterol esterification block”,
“Niemann-Pick disease, subacute juvenile form”, “lipoid histiocytosis”,
“lipoid histiocytosis (classical phosphatide)”, “sphingomyelin
lipidosis”, “sphingomyelin/cholesterol lipidosis”, “sphingomyelinase
deficiency disease”, “type A Niemann-Pick disease”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0001982" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0001982</a>”
</td>
<td style="text-align:left;">
c(“DOID:14504”, “EFO:1001380”, “GARD:13334”, “MESH:D009542”,
“NANDO:2200561”, “NCIT:C61269”, “SCTID:58459009”, “UMLS:C0028064”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NULL
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0011873”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease, type C2”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Niemann-Pick disease type C2 is a rare metabolic condition that affects
many different parts of the body. Although signs and symptoms can
develop at any age (infancy through adulthood), most affected people
develop features of the condition during childhood. Neimann-Pick disease
type C2 may be characterized by ataxia (difficulty coordinating
movements), vertical supranuclear gaze palsy (inability to move the eyes
vertically), poor muscle tone, hepatosplenomegaly (enlarged liver and
spleen), interstitial lung disease, intellectual decline, seizures,
speech problems, and difficulty swallowing. Niemann-Pick disease type C2
is caused by changes (mutations) in the NPC2 gene and is inherited in an
autosomal recessive manner. There is, unfortunately, no cure for
Niemann-Pick disease type C2. Treatment is based on the signs and
symptoms present in each person.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“NPC2”, “Niemann-PICK disease, type C2”, “Niemann-Pick disease type
C2”, “Niemann-Pick disease, type C2”, “type C2 Niemann-Pick disease”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0011873" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0011873</a>”
</td>
<td style="text-align:left;">
c(“DOID:0070114”, “GARD:3992”, “MESH:C536119”, “NCIT:C126865”,
“OMIM:607625”, “UMLS:C1843366”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“NPC is a complex lipid storage disease mainly characterized by the
accumulation of unesterified cholesterol in the late endosomal/lysosomal
compartment.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“NPC”, “Niemann Pick Disease Type C”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0018982" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0018982</a>”
</td>
<td style="text-align:left;">
c(“GARD:7207”, “MESH:D052556”, “NANDO:1200063”, “NORD:1509”,
“Orphanet:646”, “SCTID:66751000”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NULL
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0011871”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type B”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Niemann-Pick disease type B is a mild subtype of Niemann-Pick disease,
an autosomal recessive lysosomal disease, and is characterized
clinically by onset in childhood with hepatosplenomegaly, growth
retardation, and lung disorders such as infections and dyspnea”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Niemann Pick disease type B”, “Niemann-PICK disease, type B”,
“Niemann-Pick disease, Intermediate, with visceral involvement and rapid
progression”, “Niemann-Pick disease, type E”, “Niemann-Pick disease,
type F”, “type B Niemann-Pick disease”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0011871" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0011871</a>”
</td>
<td style="text-align:left;">
c(“DOID:0070112”, “GARD:10729”, “ICD10CM:E75.241”, “MESH:D052537”,
“NANDO:1200062”, “NANDO:2201207”, “NCIT:C126866”, “OMIM:607616”,
“Orphanet:77293”, “SCTID:39390005”, “UMLS:C0268243”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0100464”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“acid sphingomyelinase deficiency”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“An autosomal recessive lysosomal disease caused by biallelic loss of
function variants in the SMPD1 gene. Clinical symptoms in affected
individuals occur along a continuum. At the severe end of the spectrum
are individuals historically diagnosed with Niemann-Pick disease type A
(the neurovisceral form), which is characterized by hepatosplenomegaly
with rapid neurological deterioration leading to death in the first few
years of life. At the milder end of the spectrum are individuals
historically diagnosed with Niemann-Pick disease type B, a later-onset,
chronic visceral form, characterized by progressive visceral organ
symptoms including hepatosplenomegaly and pulmonary insufficiency, and
survival into adulthood. In addition, some affected individuals present
with an intermediate phenotype, Niemann-Pick disease type A/B.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0100464" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0100464</a>”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NULL
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0009757”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease, type C1”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Type C Niemann-Pick disease associated with a mutation in the gene
NPC1, encoding Niemann-Pick C1 protein.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“NPC1”, “Niemann-PICK disease, type C1”, “Niemann-Pick disease type
C1”, “Niemann-Pick disease with cholesterol esterification block”,
“Niemann-Pick disease without sphingomyelinase deficiency”,
“Niemann-Pick disease, chronic neuronopathic form”, “Niemann-Pick
disease, nova Scotian type”, “Niemann-Pick disease, subacute juvenile
form”, “Niemann-Pick disease, type C”, “Niemann-Pick disease, type C1”,
“Niemann-Pick disease, type D”, “neurovisceral storage disease with
vertical supranuclear ophthalmoplegia”, “type C1 Niemann-Pick disease”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009757" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009757</a>”
</td>
<td style="text-align:left;">
c(“DOID:0070113”, “NCIT:C126864”, “OMIM:257220”, “SCTID:18927009”,
“UMLS:C3179455”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0009756”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type A”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Niemann-Pick disease type A is a very severe subtype of Niemann-Pick
disease, an autosomal recessive lysosomal disease, and is characterized
clinically by onset in infancy or early childhood with failure to
thrive, hepatosplenomegaly, and rapidly progressive neurodegenerative
disorders.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Niemann-PICK disease, type A”, “Niemann-Pick disease, Intermediate,
protracted neurovisceral”, “sphingomyelin lipidosis”, “sphingomyelinase
deficiency”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009756" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009756</a>”
</td>
<td style="text-align:left;">
c(“DOID:0070111”, “GARD:7206”, “MESH:D052536”, “NANDO:1200061”,
“NANDO:2201206”, “NCIT:C126561”, “OMIM:257200”, “Orphanet:77292”,
“SCTID:52165006”, “UMLS:C0268242”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016306”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C, severe perinatal form”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016306" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016306</a>”
</td>
<td style="text-align:left;">
c(“GARD:20504”, “Orphanet:216972”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016307”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C, severe early infantile neurologic onset”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016307" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016307</a>”
</td>
<td style="text-align:left;">
c(“GARD:20505”, “Orphanet:216975”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016308”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C, late infantile neurologic onset”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016308" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016308</a>”
</td>
<td style="text-align:left;">
c(“GARD:20506”, “Orphanet:216978”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016310”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C, adult neurologic onset”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016310" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016310</a>”
</td>
<td style="text-align:left;">
c(“GARD:20508”, “NANDO:1200065”, “NANDO:2201209”, “Orphanet:216986”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0016309”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type C, juvenile neurologic onset”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Niemann-Pick disease type C, classic form”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0016309" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0016309</a>”
</td>
<td style="text-align:left;">
c(“GARD:20507”, “Orphanet:216981”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0020384”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Niemann-Pick disease type E”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Niemann-Pick disease, type E is a poorly defined adult-onset and
non-neuronopathic form of Niemann-Pick disease.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0020384" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0020384</a>”
</td>
<td style="text-align:left;">
c(“Orphanet:99022”, “SCTID:73399005”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NULL
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0850058”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“chronic neurovisceral acid sphingomyelinase deficiency”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0850058" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0850058</a>”
</td>
<td style="text-align:left;">
c(“GARD:22456”, “Orphanet:618891”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NULL
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“NPC1”
</td>
<td style="text-align:left;">
“NPC1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Niemann-Pick disease, type C1”, “SLC65A1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000141458”, “OMIM:607623”)
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
<td style="text-align:left;">
“NPC intracellular cholesterol transporter 1”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“NPC2”
</td>
<td style="text-align:left;">
“NPC2”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
c(“EDDM1”, “HE1”, “NP-C2”, “Niemann-Pick disease, type C2”, “epididymal
protein 1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000119655”, “OMIM:601015”)
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
<td style="text-align:left;">
“NPC intracellular cholesterol transporter 2”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:11120”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“SMPD1”
</td>
<td style="text-align:left;">
“SMPD1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
c(“ASM”, “Niemann-Pick type A/B”, “acid sphingomyelinase”,
“sphingomyelin phosphodiesterase 1, acid lysosomal”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000166311”, “OMIM:607608”)
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
<td style="text-align:left;">
“sphingomyelin phosphodiesterase 1”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 29 of 29 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
from
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
to
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
predicate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
agent_type
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_level
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
aggregator_knowledge_source (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
primary_knowledge_source
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
original_object
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0011873”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:869fd120-310a-4482-8e77-3016a318325b" class="uri">urn:uuid:869fd120-310a-4482-8e77-3016a318325b</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
“MONDO:0011871”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0100464”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:5d8b3ff4-8ec8-44bc-8e76-46fff5ad11cd" class="uri">urn:uuid:5d8b3ff4-8ec8-44bc-8e76-46fff5ad11cd</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0009757”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:fbebb7de-1cb9-4f90-932a-46819db221e8" class="uri">urn:uuid:fbebb7de-1cb9-4f90-932a-46819db221e8</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
“MONDO:0009756”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0100464”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:3ca10a53-3a7d-42cb-b9b2-2278ef8504ba" class="uri">urn:uuid:3ca10a53-3a7d-42cb-b9b2-2278ef8504ba</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0016306”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:4df66b1f-981b-4d05-a0eb-9986122673fc" class="uri">urn:uuid:4df66b1f-981b-4d05-a0eb-9986122673fc</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0016307”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:448c936b-6c74-4340-8d52-593f1835da95" class="uri">urn:uuid:448c936b-6c74-4340-8d52-593f1835da95</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0016308”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:12b0ebc5-a33b-4b05-943f-528d753c2ccd" class="uri">urn:uuid:12b0ebc5-a33b-4b05-943f-528d753c2ccd</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0016310”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:966840e8-2184-48fa-9aaf-56b47c501352" class="uri">urn:uuid:966840e8-2184-48fa-9aaf-56b47c501352</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
“MONDO:0016309”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:c88be317-99f7-4031-9aed-647ce2bbbc88" class="uri">urn:uuid:c88be317-99f7-4031-9aed-647ce2bbbc88</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“MONDO:0018982”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0001982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:3450e760-7666-43ec-819c-19d114fa9a88" class="uri">urn:uuid:3450e760-7666-43ec-819c-19d114fa9a88</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“MONDO:0020384”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0001982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:7ca6492d-0697-4a4f-8383-9c53e58add1e" class="uri">urn:uuid:7ca6492d-0697-4a4f-8383-9c53e58add1e</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“MONDO:0100464”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0001982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:a62ee5a6-011c-44b2-b48e-2f0ab595dadb" class="uri">urn:uuid:a62ee5a6-011c-44b2-b48e-2f0ab595dadb</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
“MONDO:0850058”
</td>
<td style="text-align:left;">
“biolink:subclass_of”
</td>
<td style="text-align:left;">
“MONDO:0001982”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“not_provided”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:monarchinitiative”, “infores:phenio”)
</td>
<td style="text-align:left;">
“infores:mondo”
</td>
<td style="text-align:left;">
“phenio_edges”
</td>
<td style="text-align:left;">
“<a href="urn:uuid:834c7761-1f42-454b-8726-9b85c3ab2bd5" class="uri">urn:uuid:834c7761-1f42-454b-8726-9b85c3ab2bd5</a>”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:Entity”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:causes”
</td>
<td style="text-align:left;">
“MONDO:0009757”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:medgen”, “infores:monarchinitiative”)
</td>
<td style="text-align:left;">
“infores:omim”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:6337715f-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CausalGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“OMIM:257220”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:causes”
</td>
<td style="text-align:left;">
“MONDO:0011873”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:medgen”, “infores:monarchinitiative”)
</td>
<td style="text-align:left;">
“infores:omim”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:631aaa18-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CausalGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“OMIM:607625”
</td>
</tr>
<tr>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
“HGNC:11120”
</td>
<td style="text-align:left;">
“biolink:causes”
</td>
<td style="text-align:left;">
“MONDO:0011871”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:medgen”, “infores:monarchinitiative”)
</td>
<td style="text-align:left;">
“infores:omim”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:631aaa33-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CausalGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:6609”
</td>
<td style="text-align:left;">
“OMIM:607616”
</td>
</tr>
<tr>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
“HGNC:11120”
</td>
<td style="text-align:left;">
“biolink:causes”
</td>
<td style="text-align:left;">
“MONDO:0009756”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
c(“infores:medgen”, “infores:monarchinitiative”)
</td>
<td style="text-align:left;">
“infores:omim”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:633771f1-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CausalGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:6609”
</td>
<td style="text-align:left;">
“OMIM:257200”
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016306”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9131-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“Orphanet:216972”
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016307”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9133-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“Orphanet:216975”
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016308”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9137-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“Orphanet:216978”
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016310”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9179-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“Orphanet:216986”
</td>
</tr>
<tr>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
“HGNC:7897”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016309”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9185-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:4864”
</td>
<td style="text-align:left;">
“Orphanet:216981”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016306”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9130-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“Orphanet:216972”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016307”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9132-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“Orphanet:216975”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016308”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9136-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“Orphanet:216978”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016310”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9178-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“Orphanet:216986”
</td>
</tr>
<tr>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
“HGNC:14537”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0016309”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:635d9184-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:10577”
</td>
<td style="text-align:left;">
“Orphanet:216981”
</td>
</tr>
<tr>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
“HGNC:11120”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0011871”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:6379fea2-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:6609”
</td>
<td style="text-align:left;">
“Orphanet:77293”
</td>
</tr>
<tr>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
“HGNC:11120”
</td>
<td style="text-align:left;">
“biolink:gene_associated_with_condition”
</td>
<td style="text-align:left;">
“MONDO:0009756”
</td>
<td style="text-align:left;">
“manual_agent”
</td>
<td style="text-align:left;">
“knowledge_assertion”
</td>
<td style="text-align:left;">
“monarch-kg_edges.jsonl”
</td>
<td style="text-align:left;">
“infores:monarchinitiative”
</td>
<td style="text-align:left;">
“infores:orphanet”
</td>
<td style="text-align:left;">
“hpoa_gene_to_disease_edges”
</td>
<td style="text-align:left;">
“uuid:6379fea3-400a-11ef-89e7-6fe0be41fbbf”
</td>
<td style="text-align:left;">
c(“biolink:Association”, “biolink:CorrelatedGeneToDiseaseAssociation”,
“biolink:Entity”, “biolink:EntityToDiseaseAssociationMixin”,
“biolink:EntityToFeatureOrDiseaseQualifiersMixin”,
“biolink:EntityToPhenotypicFeatureAssociationMixin”,
“biolink:FrequencyQualifierMixin”, “biolink:FrequencyQuantifier”,
“biolink:GeneToDiseaseAssociation”,
“biolink:GeneToDiseaseOrPhenotypicFeatureAssociation”,
“biolink:GeneToEntityAssociationMixin”,
“biolink:RelationshipQuantifier”)
</td>
<td style="text-align:left;">
“NCBIGene:6609”
</td>
<td style="text-align:left;">
“Orphanet:77292”
</td>
</tr>
</tbody>
</table>
</div>
</details><p><br></p>
<p>Backed by <code>tidygraph</code> and <code>igraph</code>,
<code>monarchr</code> is part of a larger ecosystem of graph analyses in
R:</p>
<ul>
<li>Graph joins, filters, and other manipulation: <a href="https://tidygraph.data-imaginist.com/" class="external-link"><code>tidygraph</code></a>
</li>
<li>Visualization:
<ul>
<li>ggplot-based: <a href="https://ggraph.data-imaginist.com/" class="external-link"><code>ggraph</code></a>,</li>
<li>Interactive: <a href="https://datastorm-open.github.io/visNetwork/" class="external-link"><code>visNetwork</code></a>,
<a href="https://bwlewis.github.io/rthreejs/" class="external-link"><code>threejs</code></a>
</li>
</ul>
</li>
<li>Exploratory analyses: <a href="https://r-ega.net/" class="external-link"><code>EGAnet</code></a>
</li>
<li>Graph algorithms:
<ul>
<li>Various: <a href="https://r.igraph.org/reference/index.html" class="external-link"><code>igraph</code></a>,
<a href="https://cran.r-project.org/web/packages/migraph/index.html" class="external-link"><code>migraph</code></a>,
<a href="https://github.com/schochastics/netrankr/" class="external-link"><code>netrankr</code></a>
</li>
<li>Matching: <a href="https://cran.r-project.org/web/packages/iGraphMatch/vignettes/iGraphMatch.html" class="external-link"><code>iGraphMatch</code></a>
</li>
<li>Graph neural networks: <a href="https://rpubs.com/giancarlo_vercellino/spinner" class="external-link"><code>spinner</code></a>
</li>
<li>Clustering: <a href="https://cran.r-project.org/web/packages/clustAnalytics/index.html" class="external-link"><code>clustAnalytics</code></a>
</li>
<li>Community detection: <a href="https://github.com/kharchenkolab/leidenAlg" class="external-link"><code>leidenAlg</code></a>
</li>
<li>Link prediction: <a href="https://cran.r-project.org/web/packages/linkprediction/index.html" class="external-link"><code>linkprediction</code></a>
</li>
</ul>
</li>
<li>Other:
<ul>
<li>Particle/force-based simulation: <a href="https://github.com/thomasp85/particles" class="external-link"><code>particles</code></a>
</li>
<li>… others in the reverse imports for <a href="https://cran.r-project.org/web/packages/igraph/index.html" class="external-link"><code>igraph</code></a>
</li>
</ul>
</li>
</ul>
</div>
<div class="section level2">
<h2 id="engines">Engines<a class="anchor" aria-label="anchor" href="#engines"></a>
</h2>
<p>Two kinds of engines are supported, providing largely the same
functionality: one supporting access to <a href="https://neo4j.com/" class="external-link">Neo4j graph databases</a>, and another
supporting file-based access to TSV-formatted <code>.tar.gz</code> KGs
hosted at <a href="https://kghub.io" class="external-link">kghub.io</a>. Regardless of source,
KGs must conform to the <a href="https://github.com/biolink/kgx/blob/master/specification/kgx-format.md" class="external-link">KGX
specification</a>.</p>
<p>The most accessible engine provided by <code>monarchr</code> is the
<code><a href="../reference/monarch_engine.html">monarch_engine()</a></code>, designed specifically to access the
cloud-hosted <a href="https://monarchiniatiative.org" class="external-link">Monarch
Initiative</a> KG.</p>
<div class="sourceCode" id="cb6"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">monarch</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/monarch_engine.html">monarch_engine</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p>Using another Neo4j database via <code><a href="../reference/neo4j_engine.html">neo4j_engine()</a></code> is as
simple as specifying a Neo4j <a href="https://neo4j.com/docs/operations-manual/current/configuration/ports/" class="external-link">Bolt</a>
endpoint, and username and password if required. This may be used to
host your own Neo4j database for efficient KG access, based on the
Monarch Neo4j <a href="https://github.com/monarch-initiative/monarch-neo4j" class="external-link">docker
deployment</a>.</p>
<div class="sourceCode" id="cb7"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">neodb</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/neo4j_engine.html">neo4j_engine</a></span><span class="op">(</span><span class="st">"http://localhost:7687"</span>, username <span class="op">=</span> <span class="st">"user"</span>, password <span class="op">=</span> <span class="st">"pass"</span><span class="op">)</span></span></code></pre></div>
<p>Finally, the <code><a href="../reference/file_engine.html">file_engine()</a></code> provides KG access to
KGX-formatted, <code>.tar.gz</code> files containing tab-separated
<code>*_nodes.tsv</code> and <code>*_edges.tsv</code> files. The package
comes bundled with a version of the MONDO ontology KG for demonstration
purposes. You can also provide a local file path, or a URL to a remote
file, which will be downloaded to the current working directory.</p>
<div class="sourceCode" id="cb8"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">filename</span> <span class="op">&lt;-</span> <span class="fu"><a href="https://rdrr.io/r/base/system.file.html" class="external-link">system.file</a></span><span class="op">(</span><span class="st">"extdata"</span>, <span class="st">"mondo_kgx_tsv.tar.gz"</span>, package <span class="op">=</span> <span class="st">"monarchr"</span><span class="op">)</span></span>
<span><span class="co"># filename &lt;- "https://kghub.io/kg-obo/mondo/2024-03-04/mondo_kgx_tsv.tar.gz"</span></span>
<span></span>
<span><span class="va">mondo</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/file_engine.html">file_engine</a></span><span class="op">(</span><span class="va">filename</span><span class="op">)</span></span></code></pre></div>
</div>
<div class="section level2">
<h2 id="fetching-nodes">Fetching Nodes<a class="anchor" aria-label="anchor" href="#fetching-nodes"></a>
</h2>
<p>Engines support a <code><a href="../reference/fetch_nodes.html">fetch_nodes()</a></code> function for pulling
nodes by specified criteria. It returns a <code>tidygraph</code>-based
graph, but only nodes, not any edges that may connect them. These can be
specified as a set of IDs:</p>
<div class="sourceCode" id="cb9"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">g</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MONDO:0009061"</span>, <span class="st">"HGNC:1884"</span><span class="op">)</span><span class="op">)</span></span>
<span></span>
<span><span class="va">g</span></span></code></pre></div>
<pre>Graph with 2 nodes and 0 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 2 of 2 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“HGNC:1884”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“CFTR”
</td>
<td style="text-align:left;">
“CFTR”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
c(“ABC35”, “ABCC7”, “ATP-binding cassette sub-family C, member 7”, “CF”,
“CFTR/MRP”, “MRP7”, “TNR-CFTR”, “cystic fibrosis transmembrane
conductance regulator, ATP-binding cassette (sub-family C, member 7)”,
“dJ760C5.1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000001626”, “OMIM:602421”)
</td>
<td style="text-align:left;">
“CF transmembrane conductance regulator”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
<td style="text-align:left;">
NA
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0009061”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis”
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis (CF) is a genetic disorder characterized by the
production of sweat with a high salt content and mucus secretions with
an abnormal viscosity.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CF”, “cystic fibrosis”, “cystic fibrosis lung disease, modifier of”,
“mucoviscidosis”, “pseudomonas aeruginosa, susceptibility to chronic
infection by, in cystic fibrosis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
c(“DOID:1485”, “GARD:6233”, “ICD10CM:E84”, “ICD9:277.0”, “MESH:D003550”,
“MedDRA:10011762”, “NANDO:1200922”, “NANDO:1201021”, “NANDO:2100035”,
“NANDO:2200205”, “NCIT:C2975”, “NORD:1026”, “OMIM:219700”,
“Orphanet:586”, “SCTID:190905008”, “UMLS:C0010674”,
“icd11.foundation:514403112”)
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
NA
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009061" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009061</a>”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 0 of 0 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;"><tbody><tr></tr></tbody></table>
</div>
</details><p><br> Nodes can also be fetched in bulk. For example, we can fetch
all of the nodes where <code>in_taxon_label</code> equals
<code>"Homo sapiens"</code>. For demonstration purposes, we’ll limit the
number of results to 20 (ordered by node ID).</p>
<div class="sourceCode" id="cb10"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">h_sapiens_nodes</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span><span class="va">in_taxon_label</span> <span class="op">==</span> <span class="st">"Homo sapiens"</span>, limit <span class="op">=</span> <span class="fl">20</span><span class="op">)</span></span>
<span><span class="va">h_sapiens_nodes</span></span></code></pre></div>
<pre>Graph with 20 nodes and 0 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 20 of 20 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
has_gene (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“CA10014591”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001754.5:c.110G&gt;T”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10014591”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:10471”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA10549330”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001142805.2:c.808G&gt;A”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10549330”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:11055”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA10549339”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001142805.2:c.880A&gt;C”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10549339”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:11055”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA10549367”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001142805.2:c.942C&gt;G”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10549367”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:11055”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA10549369”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001142805.2:c.952G&gt;A”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10549369”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:11055”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA10602335”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_000277.1:c.971T&gt;A”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA10602335”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532270”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001270448.2:c.1573del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532270”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:92”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532272”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001270448.2:c.1216_1220del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532272”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:92”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532470”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.463del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532470”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532474”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_000138.5:c.2034_2052del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532474”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:3603”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532533”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.912+16T&gt;A”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532533”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532534”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.1200-3T&gt;G”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532534”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532543”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.706+5G&gt;A”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532543”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532590”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.543_545del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532590”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139532728”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_000261.2:c.1187_1189dup”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139532728”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:7610”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139533026”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.843-6T&gt;C”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139533026”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139533037”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001270448.2:c.769_770insT”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139533037”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:92”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139533052”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001318054.2:c.-145_-144delinsTT”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139533052”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:5173”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139768925”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_001354304.2:c.1176dup”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139768925”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:8582”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“CA1139770890”
</td>
<td style="text-align:left;">
“biolink:BiologicalEntity”
</td>
<td style="text-align:left;">
“NM_000261.2:c.375del”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Entity”, “biolink:GenomicEntity”,
“biolink:NamedThing”, “biolink:OntologyClass”,
“biolink:PhysicalEssence”, “biolink:PhysicalEssenceOrOccurrent”,
“biolink:SequenceVariant”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“CA1139770890”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC:7610”
</td>
<td style="text-align:left;">
“clingen_variant_nodes”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 0 of 0 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;"><tbody><tr></tr></tbody></table>
</div>
</details><p><br></p>
<p>While the syntax does not support all R expressions (for
compatibility with database-backed engines), standard comparisons
<code>==</code>, <code>!=</code>, <code>&lt;=</code>,
<code>&gt;=</code>, <code>&lt;</code>, and <code>&gt;</code> are, as are
logical operators <code>&amp;</code>, <code>|</code>, and
<code>!</code>, and grouping with <code>()</code>s. Since many node
properties are multi-valued, we provide a special <code>%in_list%</code>
operator to allow exact match against entries. Here we fetch all the
<em>Homo sapiens</em> Genes, using <code>%in_list%</code> to account for
the multi-valued nature of node <code>category</code>.<a href="#fn1" class="footnote-ref" id="fnref1"><sup>1</sup></a></p>
<div class="sourceCode" id="cb11"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">human_genes</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span><span class="va">in_taxon_label</span> <span class="op">==</span> <span class="st">"Homo sapiens"</span> <span class="op">&amp;</span> <span class="st">"biolink:Gene"</span> <span class="op"><a href="../reference/grapes-in_list-grapes.html">%in_list%</a></span> <span class="va">category</span>,</span>
<span>                            limit <span class="op">=</span> <span class="fl">20</span><span class="op">)</span></span>
<span></span>
<span><span class="va">human_genes</span></span></code></pre></div>
<pre>Graph with 20 nodes and 0 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 20 of 20 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
symbol
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon_label
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
full_name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
in_taxon (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“HGNC:100”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“ASIC1”
</td>
<td style="text-align:left;">
“ASIC1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“ACCN2”, “BNaC2”, “acid sensing (proton gated) ion channel 1”,
“acid-sensing (proton-gated) ion channel 1”, “amiloride-sensitive cation
channel 2, neuronal”, “hBNaC2”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“acid sensing ion channel subunit 1”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000110881”, “OMIM:602866”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10000”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RGS4”
</td>
<td style="text-align:left;">
“RGS4”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“SCZD9”, “regulator of G-protein signaling 4”, “regulator of G-protein
signalling 4”, “schizophrenia disorder 9”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“regulator of G protein signaling 4”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000117152”, “OMIM:602516”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10001”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RGS5”
</td>
<td style="text-align:left;">
“RGS5”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“regulator of G-protein signaling 5”, “regulator of G-protein
signalling 5”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“regulator of G protein signaling 5”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000143248”, “OMIM:603276”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10002”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RGS6”
</td>
<td style="text-align:left;">
“RGS6”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“regulator of G-protein signaling 6”, “regulator of G-protein
signalling 6”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“regulator of G protein signaling 6”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000182732”, “OMIM:603894”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10003”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RGS7”
</td>
<td style="text-align:left;">
“RGS7”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“regulator of G-protein signaling 7”, “regulator of G-protein
signalling 7”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“regulator of G protein signaling 7”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000182901”, “OMIM:602517”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10004”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RGS9”
</td>
<td style="text-align:left;">
“RGS9”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“MGC111763”, “MGC26458”, “PERRS”, “RGS9L”, “regulator of G protein
signalling 9”, “regulator of G protein signalling 9L”, “regulator of
G-protein signaling 9”, “regulator of G-protein signaling 9L”,
“regulator of G-protein signalling 9”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“regulator of G protein signaling 9”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000108370”, “OMIM:604067”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10006”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHAG”
</td>
<td style="text-align:left;">
“RHAG”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Ammonium transporter Rh type A”, “CD241”, “RH50A”, “Rhesus blood
group-associated glycoprotein”, “SLC42A1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“Rh associated glycoprotein”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000112077”, “OMIM:180297”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10007”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHBDL1”
</td>
<td style="text-align:left;">
“RHBDL1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“RHBDL”, “RRP”, “rhomboid (veinlet, Drosophila)-like”, “rhomboid,
veinlet-like 1 (Drosophila)”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“rhomboid like 1”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000103269”, “OMIM:603264”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10008”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHCE”
</td>
<td style="text-align:left;">
“RHCE”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CD240CE”, “RH”, “Rhesus blood group, CcEe antigens”, “SLC42A4”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“Rh blood group CcEe antigens”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000188672”, “OMIM:111700”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10009”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHD”
</td>
<td style="text-align:left;">
“RHD”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CD240D”, “DIIIc”, “RH”, “Rh30a”, “Rh4”, “RhII”, “RhPI”, “Rhesus blood
group, D antigen”, “SLC42A5”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“Rh blood group D antigen”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000187010”, “OMIM:111680”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:1001”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“BCL6”
</td>
<td style="text-align:left;">
“BCL6”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“B cell CLL/lymphoma 6”, “BCL5”, “BCL6, transcription repressor”,
“BCL6A”, “LAZ3”, “ZBTB27”, “ZNF51”, “zinc finger protein 51”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“BCL6 transcription repressor”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000113916”, “OMIM:109565”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10010”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHEBP1”
</td>
<td style="text-align:left;">
“RHEBP1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“RHEB”, “RHEB1”, “Ras homolog enriched in brain 1”, “Ras-homolog
enriched in brain pseudogene 1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“RHEB pseudogene 1”
</td>
<td style="text-align:left;">
“ENSEMBL:ENSG00000229927”
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10011”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHEB”
</td>
<td style="text-align:left;">
“RHEB”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“RHEB2”, “Ras homolog enriched in brain”, “Ras homolog enriched in
brain 2”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“Ras homolog, mTORC1 binding”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000106615”, “OMIM:601293”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10012”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RHO”
</td>
<td style="text-align:left;">
“RHO”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CSNBAD1”, “OPN2”, “RP4”, “opsin 2, rod pigment”, “retinitis
pigmentosa 4, autosomal dominant”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“rhodopsin”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000163914”, “OMIM:180380”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10013”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“GRK1”
</td>
<td style="text-align:left;">
“GRK1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“GPRK1”, “RHOK”, “RK”, “rhodopsin kinase”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“G protein-coupled receptor kinase 1”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000185974”, “OMIM:180381”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10017”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RIT2”
</td>
<td style="text-align:left;">
“RIT2”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“RIBA”, “RIN”, “Ric (Drosophila)-like, expressed in neurons”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“Ras like without CAAX 2”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000152214”, “OMIM:609592”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10018”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RING1”
</td>
<td style="text-align:left;">
“RING1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
“RNF1”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“ring finger protein 1”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000204227”, “OMIM:602045”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10019”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RIPK1”
</td>
<td style="text-align:left;">
“RIPK1”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“RIP”, “RIP-1”, “RIP1”, “receptor (TNFRSF)-interacting
serine-threonine kinase 1”, “receptor-interacting protein kinase 1”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“receptor interacting serine/threonine kinase 1”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000137275”, “OMIM:603453”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:1002”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“BCL6B”
</td>
<td style="text-align:left;">
“BCL6B”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“B cell CLL/lymphoma 6B”, “B-cell CLL/lymphoma 6, member B”, “B-cell
CLL/lymphoma 6, member B (zinc finger protein)”, “BAZF”, “BCL6B,
transcription repressor”, “ZBTB28”, “ZNF62”, “zinc finger protein 62”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“BCL6B transcription repressor”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000161940”, “OMIM:608992”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HGNC:10020”
</td>
<td style="text-align:left;">
“biolink:Gene”
</td>
<td style="text-align:left;">
“RIPK2”
</td>
<td style="text-align:left;">
“RIPK2”
</td>
<td style="text-align:left;">
“Homo sapiens”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CARD3”, “CARDIAK”, “RICK”, “RIP2”, “receptor-interacting
serine-threonine kinase 2”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”,
“biolink:ChemicalEntityOrGeneOrGeneProduct”, “biolink:Entity”,
“biolink:Gene”, “biolink:GeneOrGeneProduct”, “biolink:GenomicEntity”,
“biolink:MacromolecularMachineMixin”, “biolink:NamedThing”,
“biolink:OntologyClass”, “biolink:PhysicalEssence”,
“biolink:PhysicalEssenceOrOccurrent”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“receptor interacting serine/threonine kinase 2”
</td>
<td style="text-align:left;">
c(“ENSEMBL:ENSG00000104312”, “OMIM:603455”)
</td>
<td style="text-align:left;">
“NCBITaxon:9606”
</td>
<td style="text-align:left;">
“HGNC”
</td>
<td style="text-align:left;">
“hgnc_gene_nodes”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 0 of 0 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;"><tbody><tr></tr></tbody></table>
</div>
</details><p><br></p>
<p>A convenience <code>%~%</code> operator provides regular-expression
matching against node properties (but not multi-valued ones like
<code>category</code>).</p>
<div class="sourceCode" id="cb12"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">fibrosis_disease_matches</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span><span class="va">description</span> <span class="op"><a href="../reference/grapes-twiddle-grapes.html">%~%</a></span> <span class="st">".*[fF]ibrosis.*"</span> <span class="op">&amp;</span> <span class="st">"biolink:Disease"</span> <span class="op"><a href="../reference/grapes-in_list-grapes.html">%in_list%</a></span> <span class="va">category</span>,</span>
<span>                            limit <span class="op">=</span> <span class="fl">20</span><span class="op">)</span></span>
<span><span class="va">fibrosis_disease_matches</span></span></code></pre></div>
<pre>Graph with 20 nodes and 0 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 20 of 20 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0000494”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“renal fibrosis”
</td>
<td style="text-align:left;min-width: 300px; ">
“A final common manifestation of a wide variety of chronic kidney
diseases characterized by glomerulosclerosis and tubulointerstitial
fibrosis.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0000494" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0000494</a>”
</td>
<td style="text-align:left;">
c(“DOID:0050855”, “EFO:1001517”, “SCTID:197660000”, “UMLS:C0151650”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0001003”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“pneumoconiosis due to talc”
</td>
<td style="text-align:left;min-width: 300px; ">
“Pneumoconiosis caused by exposure to talc. It is characterized by
fibrosis and granulomatous changes in the lung parenchyma. Chest x-rays
reveal diffuse lung opacities and pleural abnormalities.”
</td>
<td style="text-align:left;min-width: 300px; ">
“talc pneumoconiosis”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0001003" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0001003</a>”
</td>
<td style="text-align:left;">
c(“DOID:10329”, “ICD9:502”, “NCIT:C27026”, “SCTID:73144008”,
“UMLS:C0238377”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0001540”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“bagassosis”
</td>
<td style="text-align:left;min-width: 300px; ">
“An occupational lung disorder caused by inhalation of bagasse dust. In
the acute phase, it manifests as cough, dyspnea, fever, chills, and
weakness. Chronic exposure may lead to interstitial lung fibrosis.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“bagasse extrinsic allergic alveolitis”, “bagasse workers lung”,
“extrinsic allergic alveolitis from bagasse”, “sugar cane worker
pneumonitis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0001540" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0001540</a>”
</td>
<td style="text-align:left;">
c(“DOID:12522”, “ICD10CM:J67.1”, “ICD9:495.1”, “MESH:D011009”,
“NCIT:C34409”, “SCTID:67242002”, “UMLS:C0004681”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0001684”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“exocrine pancreatic insufficiency”
</td>
<td style="text-align:left;min-width: 300px; ">
“Inability of the exocrine pancreas to produce and secrete an adequate
amount of digestive enzymes into the small intestine. Patients present
with symptoms of malabsorption syndrome, abdominal discomfort, and
bloating. Causes include chronic pancreatitis, cystic fibrosis, and
autoimmune disorders.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“exocrine pancreas insufficiency”, “exocrine pancreatic
insufficiency”, “pancreatic insufficiency”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0001684" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0001684</a>”
</td>
<td style="text-align:left;">
c(“DOID:13316”, “ICD10CM:K86.81”, “ICD9:577.8”, “MESH:D010188”,
“NCIT:C84316”, “SCTID:47367009”, “UMLS:C0267963”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0002200”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“eccrine mixed tumor of skin”
</td>
<td style="text-align:left;min-width: 300px; ">
“A rare, benign, slow-growing and painless neoplasm of sweat glands. It
usually arises in the head and neck. It is characterized by the presence
of a mesenchymal chondroid stroma, fibrosis, and epithelial structures.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“benign mixed tumor of skin”, “benign mixed tumor of skin (chondroid
syringoma)”, “benign mixed tumor of the skin”, “benign mixed tumor of
the skin (chondroid syringoma)”, “benign mixed tumour of skin”, “benign
mixed tumour of skin (chondroid syringoma)”, “benign mixed tumour of the
skin”, “benign mixed tumour of the skin (chondroid syringoma)”,
“chondroid syringoma”, “eccrine mixed tumor”, “eccrine mixed tumor
(morphologic abnormality)”, “eccrine mixed tumour”, “eccrine mixed
tumour (morphologic abnormality)”, “eccrine sweat gland mixed neoplasm”,
“mixed eccrine neoplasm of the skin”, “mixed tumor of the skin
(chondroid syringoma)”, “mixed tumour of the skin (chondroid
syringoma)”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0002200" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0002200</a>”
</td>
<td style="text-align:left;">
c(“DOID:2079”, “EFO:1000385”, “NCIT:C4474”, “SCTID:254720009”,
“UMLS:C0346026”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0003246”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“sclerosing hepatic carcinoma”
</td>
<td style="text-align:left;min-width: 300px; ">
“An uncommon type of hepatocelluar carcinoma, morphologically
characterized by significant fibrosis around the sinusoid-like spaces
and atrophy of the tumor trabeculae.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“scirrhous hepatocellular cancer”, “scirrhous hepatocellular
carcinoma”, “sclerosing hepatic carcinoma”, “sclerosing hepatocellular
carcinoma”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0003246" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0003246</a>”
</td>
<td style="text-align:left;">
c(“DOID:5026”, “ICDO:8172/3”, “NCIT:C27388”, “UMLS:C1266018”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0003643”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“giant hemangioma”
</td>
<td style="text-align:left;min-width: 300px; ">
“A cavernous hemangioma characterized by the presence of hylanized
vascular channels and is often associated with the presence of
calcifications, fibrosis, and hemorrhage.”
</td>
<td style="text-align:left;min-width: 300px; ">
“giant hemangioma”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0003643" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0003643</a>”
</td>
<td style="text-align:left;">
c(“DOID:5774”, “NCIT:C27777”, “UMLS:C1333817”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0003701”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“thyroid gland diffuse sclerosing papillary carcinoma”
</td>
<td style="text-align:left;min-width: 300px; ">
“A morphologic variant of papillary carcinoma of the thyroid gland that
more often affects young patients and commonly metastasizing to the
lungs. It is characterized by a diffuse infiltration of the thyroid
gland by malignant follicular cells, squamous metaplasia, stromal
fibrosis, and lymphocytic infiltration.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“nonencapsulated sclerosing adenocarcinoma”, “nonencapsulated
sclerosing carcinoma”, “nonencapsulated sclerosing neoplasm”,
“nonencapsulated sclerosing papillary thyroid carcinoma”,
“nonencapsulated sclerosing tumor”, “nonencapsulated sclerosing tumour”,
“papillary carcinoma, diffuse sclerosing”, “thyroid gland diffuse
sclerosing papillary carcinoma”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0003701" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0003701</a>”
</td>
<td style="text-align:left;">
c(“DOID:5914”, “ICDO:8350/3”, “NCIT:C7427”, “UMLS:C0334330”,
“UMLS:C1321862”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0004288”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“scirrhous breast carcinoma”
</td>
<td style="text-align:left;min-width: 300px; ">
“An infiltrating ductal breast carcinoma associated with stromal
fibrosis.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“breast scirrhous carcinoma”, “infiltrating carcinoma of breast with
fibrotic Stroma”, “infiltrating carcinoma of the breast with fibrotic
Stroma”, “scirrhous breast carcinoma”, “scirrhous carcinoma of breast”,
“scirrhous carcinoma of the breast”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0004288" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0004288</a>”
</td>
<td style="text-align:left;">
c(“DOID:7578”, “ICDO:8514/3”, “NCIT:C7362”, “SCTID:254839007”,
“UMLS:C0346151”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0004463”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cellular phase chronic idiopathic myelofibrosis”
</td>
<td style="text-align:left;min-width: 300px; ">
“Primary myelofibrosis characterized by bone marrow hypercellularity and
the presence of atypical megakaryocytes. There is no increase in the
percentage of myeloblasts and no significant increase in reticulin or
collagen fibrosis in the bone marrow.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“PMFPES”, “Prefibrotic/Early Primary myelofibrosis”, “chronic
idiopathic myelofibrosis, Prefibrotic stage”, “chronic idiopathic
myelofibrosis, cellular phase”, “primary myelofibrosis, Prefibrotic
stage”, “primary myelofibrosis, Prefibrotic/early stage”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0004463" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0004463</a>”
</td>
<td style="text-align:left;">
c(“DOID:8106”, “NCIT:C41237”, “ONCOTREE:PMFPES”, “UMLS:C1516553”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0004633”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Hodgkin’s lymphoma, mixed cellularity”
</td>
<td style="text-align:left;min-width: 300px; ">
“A subtype of classical Hodgkin lymphoma with scattered Reed-Sternberg
and Hodgkin cells in a diffuse or vaguely nodular mixed inflammatory
background without nodular sclerosing fibrosis. (WHO, 2008)”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Hodgkin lymphoma, mixed cellularity”, “Hodgkin’s disease mixed
cellularity”, “Hodgkin’s disease, mixed cellularity”, “Hodgkin’s
disease, mixed cellularity of unspecified site”, “Hodgkin’s lymphoma
mixed cellularity”, “MCCHL”, “MCHL”, “Mixed cellularity Classic Hodgkin
lymphoma”, “classic Hodgkin lymphoma, mixed cellularity type”, “mixed
cellularity Hodgkin lymphoma”, “mixed cellularity Hodgkin’s disease”,
“mixed cellularity Hodgkin’s lymphoma”, “mixed cellularity classical
Hodgkin lymphoma”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0004633" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0004633</a>”
</td>
<td style="text-align:left;">
c(“DOID:8654”, “EFO:1002031”, “GARD:19592”, “ICD9:201.6”, “ICDO:9652/3”,
“NCIT:C3517”, “ONCOTREE:MCCHL”, “Orphanet:98844”, “SCTID:118609008”,
“icd11.foundation:39515681”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0004769”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“orbital pseudotumor”
</td>
<td style="text-align:left;min-width: 300px; ">
“A nonspecific tumor-like inflammatory lesion in the orbit of the eye.
It is usually composed of mature lymphocytes; plasma cells; macrophages;
leukocytes with varying degrees of fibrosis. Orbital pseudotumors are
often associated with inflammation of the extraocular muscles (orbital
myositis) or inflammation of the lacrimal glands (dacryoadenitis).”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“granuloma, plasma cell, orbital”, “inflammatory pseudotumor of
orbit”, “inflammatory pseudotumor of orbit proper”, “inflammatory
pseudotumor, orbital”, “inflammatory pseudotumors, orbital”, “orbital
granuloma, plasma cell”, “orbital inflammatory pseudotumor”, “orbital
inflammatory pseudotumors”, “orbital myositis”, “orbital plasma cell
granuloma”, “orbital pseudotumors”, “plasma cell granuloma, orbital”,
“pseudotumor of orbit”, “pseudotumor, inflammatory, orbital”,
“pseudotumor, orbital”, “pseudotumor, orbital inflammatory”,
“pseudotumors, orbital”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0004769" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0004769</a>”
</td>
<td style="text-align:left;">
c(“DOID:9369”, “MESH:D016727”, “SCTID:72789009”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005003”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“chronic pancreatitis”
</td>
<td style="text-align:left;min-width: 300px; ">
“A chronic inflammatory process causing damage and fibrosis of the
pancreatic parenchyma. Signs and symptoms include abdominal pain,
malabsorption and diabetes mellitus.”
</td>
<td style="text-align:left;min-width: 300px; ">
“pancreatitis, chronic”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005003" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005003</a>”
</td>
<td style="text-align:left;">
c(“EFO:0000342”, “ICD9:577.1”, “MESH:D050500”, “NCIT:C84637”,
“SCTID:235494005”, “UMLS:C0149521”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005011”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Crohn disease”
</td>
<td style="text-align:left;min-width: 300px; ">
“A gastrointestinal disorder characterized by chronic inflammation
involving all layers of the intestinal wall, noncaseating granulomas
affecting the intestinal wall and regional lymph nodes, and transmural
fibrosis. Crohn disease most commonly involves the terminal ileum; the
colon is the second most common site of involvement.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Crohn disease”, “Crohn’s disease”, “Crohn’s disease of colon”,
“Crohn’s disease of large bowel”, “granulomatous colitis”, “paediatric
Crohn’s disease”, “pediatric Crohn’s disease”, “regional enteritis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005011" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005011</a>”
</td>
<td style="text-align:left;">
c(“DOID:8778”, “EFO:0000384”, “ICD9:555.1”, “MESH:D003424”,
“NANDO:1200444”, “NANDO:1200446”, “NANDO:2200921”, “NCIT:C2965”,
“Orphanet:206”, “SCTID:34000006”, “SCTID:7620006”, “UMLS:C0010346”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005019”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“diffuse scleroderma”
</td>
<td style="text-align:left;min-width: 300px; ">
“A variant of systemic scleroderma characterized by sclerosis of the
skin, Raynaud phenomenon, and organ involvement, including pulmonary
fibrosis, renal disease, and gastrointestinal tract involvement.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“diffuse systemic sclerosis”, “systemic sclerosis, diffuse”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005019" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005019</a>”
</td>
<td style="text-align:left;">
c(“DOID:1580”, “EFO:0000404”, “GARD:7727”, “MESH:D045743”,
“NCIT:C116791”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005100”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“systemic sclerosis”
</td>
<td style="text-align:left;min-width: 300px; ">
“A chronic disorder, possibly autoimmune, marked by excessive production
of collagen which results in hardening and thickening of body tissues.
The two types of systemic scleroderma, limited cutaneous and diffuse
cutaneous are classified with focus on the extent of affected skin. A
relationship exists between the extent of skin area affected and degree
of internal organ/system involvement. Systemic scleroderma can manifest
itself in pulmonary fibrosis, Raynaud’s syndrome, digestive system
telangiectasias, renal hypertension and/or pulmonary hypertension.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“PSS (progressive systemic sclerosis)”, “SSc”, “SSc, diffuse
sclerosis”, “Scleroderma”, “Scleroderma (&amp; [systemic sclerosis])”,
“Scleroderma syndrome”, “Scleroderma, diffuse”, “Scleroderma, systemic”,
“Systemic Scleroderma”, “diffuse Scleroderma”, “diffuse sclerosis”,
“progressive systemic sclerosis”, “systemic Scleroderma”, “systemic
scleroderma”, “systemic sclerosis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005100" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005100</a>”
</td>
<td style="text-align:left;">
c(“DOID:418”, “EFO:0000717”, “GARD:9748”, “ICD9:710.1”, “MESH:D012595”,
“MedDRA:10042953”, “NANDO:1200277”, “NANDO:2200429”, “NCIT:C72070”,
“NORD:2007”, “Orphanet:90291”, “SCTID:89155008”, “UMLS:C0036421”,
“icd11.foundation:1084365812”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005219”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“breast fibrocystic disease”
</td>
<td style="text-align:left;min-width: 300px; ">
“Fibrosis associated with cyst formation in the breast parenchyma.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“benign breast disease”, “breast fibrocystic change”, “cystic disease
of breast”, “cystic disease of the breast”, “diffuse cystic mastopathy”,
“fibrocystic breast”, “fibrocystic breast changes”, “fibrocystic breast
disease”, “fibrocystic change of breast”, “fibrocystic change of the
breast”, “fibrocystic disease”, “fibrocystic disease of breast”,
“fibrocystic disease of the breast”, “fibrocystic mastopathy”, “mammary
dysplasia”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005219" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005219</a>”
</td>
<td style="text-align:left;">
c(“DOID:10354”, “EFO:0003014”, “ICD9:610.1”, “MESH:D005348”,
“NCIT:C3039”, “UMLS:C0016034”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005413”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis associated meconium ileus”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis associated meconium ileum is a thickening and
congestion of the meconium in the ileum in newborns, often the first
sign of cystic fibrosis. In cystic fibrosis, the meconium can form a
bituminous black-green mechanical obstruction in a segment of the ileum.
Beyond this there may be a few separate grey-white globular pellets.
Below this level, the bowel is a narrow and empty micro-colon. Above the
level of the obstruction, there are several loops of hypertrophied bowel
distended with fluid. No meconium is passed, and abdominal distension
and vomiting appear soon after birth. About 20% of cases of cystic
fibrosis present with meconium ileus, while approximately 20% of one
series of cases of meconium ileus did not have cystic fibrosis. The
presence of meconium ileus is not related to the severity of the cystic
fibrosis.”
</td>
<td style="text-align:left;min-width: 300px; ">
“cystic fibrosis associated meconium ileum”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005413" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005413</a>”
</td>
<td style="text-align:left;">
“EFO:0004608”
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005668”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“bird fancier’s lung”
</td>
<td style="text-align:left;min-width: 300px; ">
“Hypersensitivity granulomatous pneumonitis caused by the inhalation of
avian antigens that are present in the dust of the droppings and
feathers of many species of birds. In the acute phase it manifests as
fever, chills, dyspnea, cough, and chest tightness. Chronic exposure may
lead to interstitial lung fibrosis.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Avian hypersensitivity pneumonitis”, “bird breeder’s lung”, “bird
fancier lung”, “bird fancier’s lung”, “bird-breeder’s lung”,
“bird-fancier’s lung”, “bird-fanciers’ lung”, “pigeon breeder’s lung”,
“pigeon-breeder lung disease”, “pigeon-breeder’s lung”, “poultry
worker’s lung”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005668" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005668</a>”
</td>
<td style="text-align:left;">
c(“DOID:13891”, “GARD:16924”, “ICD10CM:J67.2”, “ICD9:495.2”,
“MESH:D001716”, “MedDRA:10004941”, “NCIT:C34425”, “Orphanet:99908”,
“SCTID:69339004”, “UMLS:C0005592”, “icd11.foundation:912113736”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0005705”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“clonorchiasis”
</td>
<td style="text-align:left;min-width: 300px; ">
“Infection of the biliary passages with clonorchis sinensis, also called
Opisthorchis sinensis. It may lead to inflammation of the biliary tract,
proliferation of biliary epithelium, progressive portal fibrosis, and
sometimes bile duct carcinoma. Extension to the liver may lead to fatty
changes and cirrhosis. (From Dorland, 27th ed)”
</td>
<td style="text-align:left;min-width: 300px; ">
“Oriental liver fluke disease”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0005705" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0005705</a>”
</td>
<td style="text-align:left;">
c(“DOID:13767”, “EFO:0007210”, “ICD10CM:B66.1”, “ICD9:121.1”,
“MESH:D003003”, “Orphanet:658917”, “SCTID:11938002”, “UMLS:C0009021”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 0 of 0 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;"><tbody><tr></tr></tbody></table>
</div>
</details><p><br></p>
</div>
<div class="section level2">
<h2 id="expanding">Expanding<a class="anchor" aria-label="anchor" href="#expanding"></a>
</h2>
<p>From a set of nodes (and possibly edges that connect them), we can
“expand” by fetching their collective neighborhood from the KG engine.
Here we fetch 10 Gene nodes, and in the first call to
<code><a href="../reference/expand.html">expand()</a></code> fetch nodes connected by
<code>biolink:causes</code> or
<code>biolink:gene_associated_with_condition</code> predicates. We then
take the resulting graph (which contains both disease and gene nodes),
and fetch the neighborhood of nodes with category
<code>biolink:PhenotypicFeature</code>. Since there are many such
results, we add a <code>limit</code> for the visualization.<a href="#fn2" class="footnote-ref" id="fnref2"><sup>2</sup></a></p>
<div class="sourceCode" id="cb13"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># MONDO:0018982, MONDO:0011871: Neimann-Pick type B and C</span></span>
<span></span>
<span><span class="va">g</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MONDO:0018982"</span>, <span class="st">"MONDO:0011871"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"biolink:causes"</span>, <span class="st">"biolink:gene_associated_with_condition"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>categories <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"biolink:PhenotypicFeature"</span><span class="op">)</span>, limit <span class="op">=</span> <span class="fl">10</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">g</span><span class="op">)</span></span></code></pre></div>
<p><img src="monarchr_files/figure-html/unnamed-chunk-10-1.png" width="960">
We can limit the expansion based on several factors, which are combined
in an “and” fashion:</p>
<ul>
<li>
<code>direction</code>: <code>"in</code>“, <code>"out"</code>, or
<code>"both"</code> (default): which direction edges should point from
the query graph nodes</li>
<li>
<code>predicates</code>: a character vector of edge predicates to
fetch (all if unspecified)</li>
<li>
<code>categories</code>: a character vector of node category labels
to fetch in the connected neighborhood (all if unspecified)</li>
</ul>
<p>The <code>drop_unused_query_nodes</code> parameter (which defaults to
<code>FALSE</code>) keeps only the nodes that are part of the
recently-expanded neighborhood. For example, given a query graph with
many diseases, if we try to fetch neighboring phenotypes, setting this
to <code>TRUE</code> removes those disease nodes without any
phenotypes.</p>
<p>Lastly, the <code>transitive</code> parameter (which defaults to
<code>FALSE</code>) can be used to fetch hierarchical, transitive
relationships like <code>biolink:subclass_of</code>. When using
<code>transitive = TRUE</code>, direction must be <code>"in"</code> or
<code>"out"</code>, and <code>predicates</code> must be a single entry.
The first example above illustrated this; here we explore the
super-types and sub-types of Niemann-Pick, by first fetching all of its
subtypes, and then considering the collective ancestry of those nodes in
the KG. Not all ancestors are diseases
(e.g. <code>biolink:Entity</code>), so we restrict the result set of the
second expansion.</p>
<div class="sourceCode" id="cb14"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="co"># MONDO:0001982 Niemann-Pick Disease (13 subtypes)</span></span>
<span></span>
<span><span class="va">hierarchy</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>  <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="fu"><a href="https://rdrr.io/r/base/c.html" class="external-link">c</a></span><span class="op">(</span><span class="st">"MONDO:0001982"</span><span class="op">)</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="st">"biolink:subclass_of"</span>, direction <span class="op">=</span> <span class="st">"in"</span>, transitive <span class="op">=</span> <span class="cn">TRUE</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="st">"biolink:subclass_of"</span>, direction <span class="op">=</span> <span class="st">"out"</span>, transitive <span class="op">=</span> <span class="cn">TRUE</span>, categories <span class="op">=</span> <span class="st">"biolink:Disease"</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">hierarchy</span><span class="op">)</span></span></code></pre></div>
<p><img src="monarchr_files/figure-html/unnamed-chunk-11-1.png" width="960"></p>
</div>
<div class="section level2">
<h2 id="monarch-features">Monarch Features<a class="anchor" aria-label="anchor" href="#monarch-features"></a>
</h2>
<div class="section level3">
<h3 id="searching">Searching<a class="anchor" aria-label="anchor" href="#searching"></a>
</h3>
<p>While <code><a href="../reference/fetch_nodes.html">fetch_nodes()</a></code> and <code><a href="../reference/expand.html">expand()</a></code> apply to
all types of engines, <code>monarchr</code> also exposes functionality
specific to the Monarch Initiatives’ API services. The first of these is
<code><a href="../reference/monarch_search.html">monarch_search()</a></code>, which uses the same search API as the
Monarch <a href="https://monarchinitiative.org" class="external-link">website</a>, returning a
graph with matching nodes backed by a <code><a href="../reference/monarch_engine.html">monarch_engine()</a></code>.</p>
<div class="sourceCode" id="cb15"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">cf_hits</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/monarch_search.html">monarch_search</a></span><span class="op">(</span><span class="st">"Cystic fibrosis"</span>, category <span class="op">=</span> <span class="st">"biolink:Disease"</span>, limit <span class="op">=</span> <span class="fl">5</span><span class="op">)</span></span>
<span></span>
<span><span class="va">cf_hits</span></span></code></pre></div>
<pre>Graph with 5 nodes and 0 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 5 of 5 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0009061”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis (CF) is a genetic disorder characterized by the
production of sweat with a high salt content and mucus secretions with
an abnormal viscosity.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“CF”, “cystic fibrosis”, “cystic fibrosis lung disease, modifier of”,
“mucoviscidosis”, “pseudomonas aeruginosa, susceptibility to chronic
infection by, in cystic fibrosis”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0009061" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0009061</a>”
</td>
<td style="text-align:left;">
c(“DOID:1485”, “GARD:6233”, “ICD10CM:E84”, “ICD9:277.0”, “MESH:D003550”,
“MedDRA:10011762”, “NANDO:1200922”, “NANDO:1201021”, “NANDO:2100035”,
“NANDO:2200205”, “NCIT:C2975”, “NORD:1026”, “OMIM:219700”,
“Orphanet:586”, “SCTID:190905008”, “UMLS:C0010674”,
“icd11.foundation:514403112”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:1010043”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis, non-human animal”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis that occurs in non-human animals.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_1010043" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_1010043</a>”
</td>
<td style="text-align:left;">
“OMIA:001794”
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:1010543”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis, domestic ferret”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis that occurs in domestic ferret.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_1010543" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_1010543</a>”
</td>
<td style="text-align:left;">
“OMIA:001794-9669”
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:1010544”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis, pig”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis that occurs in pig.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_1010544" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_1010544</a>”
</td>
<td style="text-align:left;">
“OMIA:001794-9823”
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:1010545”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“cystic fibrosis, sheep”
</td>
<td style="text-align:left;min-width: 300px; ">
“Cystic fibrosis that occurs in sheep.”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_1010545" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_1010545</a>”
</td>
<td style="text-align:left;">
“OMIA:001794-9940”
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 0 of 0 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;"><tbody><tr></tr></tbody></table>
</div>
</details><p><br></p>
</div>
<div class="section level3">
<h3 id="semantic-similarity">Semantic Similarity<a class="anchor" aria-label="anchor" href="#semantic-similarity"></a>
</h3>
<p>Monarch’s <a href="https://monarchinitiative.org/explore#phenotype-explorer" class="external-link">Semantic
Similarity</a> tool provides matches between given query and target
nodes, based on their semantic similarity in the KG. Here we grab a few
phenotypes of Ehlers-Danlos syndrome, classic type
(<code>"MONDO:0007522"</code>) and Marfan Syndrome
(<code>"MONDO:0007947"</code>), and we find the best matching Marfan
phenotype for each EDS phenotype. The result is returned as a graph,
containing only computed <code>"computed:best_matches"</code> edges:</p>
<div class="sourceCode" id="cb16"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">eds_phenos</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="st">"MONDO:0007522"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="st">"biolink:has_phenotype"</span>, limit <span class="op">=</span> <span class="fl">10</span><span class="op">)</span></span>
<span></span>
<span><span class="va">marfan_phenos</span> <span class="op">&lt;-</span> <span class="va">monarch</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/fetch_nodes.html">fetch_nodes</a></span><span class="op">(</span>query_ids <span class="op">=</span> <span class="st">"MONDO:0007947"</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="../reference/expand.html">expand</a></span><span class="op">(</span>predicates <span class="op">=</span> <span class="st">"biolink:has_phenotype"</span>, limit <span class="op">=</span> <span class="fl">10</span><span class="op">)</span></span>
<span></span>
<span><span class="va">sim</span> <span class="op">&lt;-</span> <span class="fu"><a href="../reference/monarch_semsim.html">monarch_semsim</a></span><span class="op">(</span><span class="va">eds_phenos</span>, <span class="va">marfan_phenos</span><span class="op">)</span></span>
<span></span>
<span><span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="va">sim</span><span class="op">)</span></span>
<span><span class="va">sim</span></span></code></pre></div>
<p><img src="monarchr_files/figure-html/unnamed-chunk-13-1.png" width="960"></p>
<pre>Graph with 18 nodes and 11 edges. Expand sections below for details.</pre>
<details><summary><b>Node Data</b>
</summary><p>
Showing 18 of 18 nodes:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
id
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
pcategory
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
name
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
description
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
synonym (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
category (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
iri
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
xref (list)
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
namespace
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
provided_by (list)
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
“MONDO:0007522”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Ehlers-Danlos syndrome, classic type”
</td>
<td style="text-align:left;min-width: 300px; ">
“Ehlers-Danlos syndrome, classic type (cEDS) is a form of Ehlers-Danlos
syndrome that affects the connective tissue and is characterized by skin
hyperextensibility, widened atrophic scars and joint hypermobility.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“EDS I”, “EDS I, formerly”, “EDS II”, “EDS II, formerly”, “EDS,
classic type”, “Ehlers Danlos syndrome, mild classic type”, “Ehlers
Danlos syndrome, mild classic type, formerly”, “Ehlers Danlos syndrome,
mitis type”, “Ehlers Danlos syndrome, mitis type, formerly”,
“Ehlers-Danlos syndrome classic type”, “Ehlers-Danlos syndrome classical
type”, “Ehlers-Danlos syndrome type 1 (formerly)”, “Ehlers-Danlos
syndrome type 2”, “Ehlers-Danlos syndrome type 2 (formerly)”,
“Ehlers-Danlos syndrome, classic type”, “Ehlers-Danlos syndrome, gravis
type”, “Ehlers-Danlos syndrome, gravis type, formerly”, “Ehlers-Danlos
syndrome, severe classic type”, “Ehlers-Danlos syndrome, severe classic
type, formerly”, “Ehlers-Danlos syndrome, type I”, “Ehlers-Danlos
syndrome, type I, formerly”, “Ehlers-Danlos syndrome, type II”,
“Ehlers-Danlos syndrome, type II, formerly”, “classic Ehlers-Danlos
syndrome”, “classical Ehlers-Danlos syndrome”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0007522" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0007522</a>”
</td>
<td style="text-align:left;">
c(“GARD:2088”, “NANDO:1200646”, “NANDO:2201256”, “Orphanet:287”,
“SCTID:715318006”, “UMLS:C0220679”, “UMLS:C4225429”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hyperextensible skin”
</td>
<td style="text-align:left;min-width: 300px; ">
“A condition in which the skin can be stretched beyond normal, and then
returns to its initial position.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Hyperelastic skin”, “Skin hyperelasticity”, “Skin
hyperextensibility”, “Stretchable skin”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000974" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000974</a>”
</td>
<td style="text-align:left;">
“UMLS:C0241074”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001027”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Soft, doughy skin”
</td>
<td style="text-align:left;min-width: 300px; ">
“A skin texture that is unusually soft (and may feel silky), and has a
malleable consistency resembling that of dough.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Soft, doughy skin”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001027" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001027</a>”
</td>
<td style="text-align:left;">
“UMLS:C1849043”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001030”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Fragile skin”
</td>
<td style="text-align:left;min-width: 300px; ">
“Skin that splits easily with minimal injury.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Fragile skin”, “Skin fragility”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001030" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001030</a>”
</td>
<td style="text-align:left;">
c(“MEDDRA:10040851”, “SNOMEDCT_US:247427007”, “UMLS:C0241181”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001065”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Striae distensae”
</td>
<td style="text-align:left;min-width: 300px; ">
“Thinned, erythematous, depressed bands of atrophic skin. Initially,
striae appear as flattened and thinned, pinkish linear regions of the
skin. Striae tend to enlarge in length and become reddish or purplish.
Later, striae tend to appear as white, depressed bands that are parallel
to the lines of skin tension. Striae distensae occur most often in areas
that have been subject to distension such as the lower back, buttocks,
thighs, breast, abdomen, and shoulders.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Purplish striae”, “Stretch marks”, “Striae”, “Striae atrophicae”,
“Striae cutis distensae”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001065" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001065</a>”
</td>
<td style="text-align:left;">
c(“MEDDRA:10040925”, “MSH:D057896”, “SNOMEDCT_US:201066002”,
“SNOMEDCT_US:201067006”, “SNOMEDCT_US:47212006”, “UMLS:C0152459”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001073”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Cigarette-paper scars”
</td>
<td style="text-align:left;min-width: 300px; ">
“Thin (atrophic) and wide scars.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cigarette paper scarring”, “Cigarette-paper scars”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001073" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001073</a>”
</td>
<td style="text-align:left;">
“UMLS:C1851828”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001075”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Atrophic scars”
</td>
<td style="text-align:left;min-width: 300px; ">
“Scars that form a depression compared to the level of the surrounding
skin because of damage to the collagen, fat or other tissues below the
skin.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Sunken or indented skin due to damage”, “Thin, atrophic scars”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001075" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001075</a>”
</td>
<td style="text-align:left;">
c(“SNOMEDCT_US:239172000”, “SNOMEDCT_US:409766009”, “UMLS:C0162154”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0002761”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Generalized joint laxity”
</td>
<td style="text-align:left;min-width: 300px; ">
“Joint hypermobility (ability of a joint to move beyond its normal range
of motion) affecting many or all joints of the body.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Generalised joint laxity”, “Hypermobility of all joints”, “Joint
laxity, generalised”, “Joint laxity, generalized”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0002761" class="external-link uri">http://purl.obolibrary.org/obo/HP_0002761</a>”
</td>
<td style="text-align:left;">
“UMLS:C1836308”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000938”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Osteopenia”
</td>
<td style="text-align:left;min-width: 300px; ">
“Osteopenia is a term to define bone density that is not normal but also
not as low as osteoporosis. By definition from the World Health
Organization osteopenia is defined by bone densitometry as a T score -1
to -2.5.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Generalised osteopenia”, “Generalized osteopenia”, “Osteopaenia”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000938" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000938</a>”
</td>
<td style="text-align:left;">
c(“MSH:D001851”, “SNOMEDCT_US:312894000”, “SNOMEDCT_US:78441005”,
“UMLS:C0029453”, “UMLS:C0747078”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001058”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Poor wound healing”
</td>
<td style="text-align:left;min-width: 300px; ">
“A reduced ability to heal cutaneous wounds.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Poor wound healing”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001058" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001058</a>”
</td>
<td style="text-align:left;">
“UMLS:C1851789”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001252”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Hypotonia”
</td>
<td style="text-align:left;min-width: 300px; ">
“Hypotonia is an abnormally low muscle tone (the amount of tension or
resistance to movement in a muscle). Even when relaxed, muscles have a
continuous and passive partial contraction which provides some
resistance to passive stretching. Hypotonia thus manifests as diminished
resistance to passive stretching. Hypotonia is not the same as muscle
weakness, although the two conditions can co-exist.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Central hypotonia”, “Low muscle tone”, “Low or weak muscle tone”,
“Muscle hypotonia”, “Muscular hypotonia”, “Peripheral hypotonia”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001252" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001252</a>”
</td>
<td style="text-align:left;">
c(“MSH:D009123”, “SNOMEDCT_US:398151007”, “SNOMEDCT_US:398152000”,
“UMLS:C0026827”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“biolink:Disease”
</td>
<td style="text-align:left;">
“Marfan syndrome”
</td>
<td style="text-align:left;min-width: 300px; ">
“A disorder of the connective tissue. Connective tissue provides
strength and flexibility to structures throughout the body such as
bones, ligaments, muscles, walls of blood vessels, and heart valves.
Marfan syndrome affects most organs and tissues, especially the
skeleton, lungs, eyes, heart, and the large blood vessel that
distributes blood from the heart to the rest of the body (the aorta). It
is caused by mutations in the FBN1 gene, which provides instructions for
making a protein called fibrillin-1. Marfan syndrome is inherited in an
autosomal dominant pattern. At least 25% of cases are due to a new (de
novo) mutation. Treatment is based on the signs and symptoms in each
person.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“MFS”, “MFS1”, “Marfan syndrome”, “Marfan syndrome type 1”, “Marfan
syndrome, type 1”, “Marfan’s syndrome”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:Disease”,
“biolink:DiseaseOrPhenotypicFeature”, “biolink:Entity”,
“biolink:NamedThing”, “biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/MONDO_0007947" class="external-link uri">http://purl.obolibrary.org/obo/MONDO_0007947</a>”
</td>
<td style="text-align:left;">
c(“DOID:14323”, “GARD:16535”, “ICD9:759.82”, “MESH:D008382”,
“MedDRA:10026829”, “NANDO:1200644”, “NANDO:2200968”, “NCIT:C34807”,
“NORD:1403”, “OMIM:154700”, “Orphanet:284963”, “Orphanet:558”,
“SCTID:19346006”, “UMLS:C0024796”, “icd11.foundation:236564145”)
</td>
<td style="text-align:left;">
“MONDO”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001377”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Limited elbow extension”
</td>
<td style="text-align:left;min-width: 300px; ">
“Limited ability to straighten the arm at the elbow joint.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Decreased elbow extension”, “Elbow limited extension”, “Limitation of
elbow extension”, “Limited elbow extension”, “Limited extension at
elbows”, “Limited forearm extension”, “Restricted elbow extension”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001377" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001377</a>”
</td>
<td style="text-align:left;">
“UMLS:C1867103”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000486”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Strabismus”
</td>
<td style="text-align:left;min-width: 300px; ">
“A misalignment of the eyes so that the visual axes deviate from
bifoveal fixation. The classification of strabismus may be based on a
number of features including the relative position of the eyes, whether
the deviation is latent or manifest, intermittent or constant,
concomitant or otherwise and according to the age of onset and the
relevance of any associated refractive error.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cross-eyed”, “Squint”, “Squint eyes”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000486" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000486</a>”
</td>
<td style="text-align:left;">
c(“MSH:D013285”, “SNOMEDCT_US:128602000”, “SNOMEDCT_US:22066006”,
“UMLS:C0038379”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0005136”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Mitral annular calcification”
</td>
<td style="text-align:left;min-width: 300px; ">
“Mitral annular calcification (MAC) results from progressive calcium
deposition along and beneath the mitral valve annulus.”
</td>
<td style="text-align:left;min-width: 300px; ">
“Premature calcification of mitral annulus”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0005136" class="external-link uri">http://purl.obolibrary.org/obo/HP_0005136</a>”
</td>
<td style="text-align:left;">
“UMLS:C1835130”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0001371”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Flexion contracture”
</td>
<td style="text-align:left;min-width: 300px; ">
“A flexion contracture is a bent (flexed) joint that cannot be
straightened actively or passively. It is thus a chronic loss of joint
motion due to structural changes in muscle, tendons, ligaments, or skin
that prevents normal movement of joints.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Flexed joint that cannot be straightened”, “Flexion contractures”,
“Flexion contractures of joints”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0001371" class="external-link uri">http://purl.obolibrary.org/obo/HP_0001371</a>”
</td>
<td style="text-align:left;">
c(“MSH:D003286”, “SNOMEDCT_US:203598005”, “SNOMEDCT_US:385522000”,
“SNOMEDCT_US:55033002”, “SNOMEDCT_US:57048009”, “SNOMEDCT_US:7890003”,
“SNOMEDCT_US:88565003”, “UMLS:C0009917”, “UMLS:C0009918”,
“UMLS:C0333068”, “UMLS:C1850530”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0003199”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Decreased muscle mass”
</td>
<td style="text-align:left;min-width: 300px; ">
NA
</td>
<td style="text-align:left;min-width: 300px; ">
“Decreased muscle mass”
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0003199" class="external-link uri">http://purl.obolibrary.org/obo/HP_0003199</a>”
</td>
<td style="text-align:left;">
“UMLS:C1837108”
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
<tr>
<td style="text-align:left;">
“HP:0000518”
</td>
<td style="text-align:left;">
“biolink:PhenotypicFeature”
</td>
<td style="text-align:left;">
“Cataract”
</td>
<td style="text-align:left;min-width: 300px; ">
“A cataract is an opacity or clouding that develops in the crystalline
lens of the eye or in its capsule.”
</td>
<td style="text-align:left;min-width: 300px; ">
c(“Cataracts”, “Clouding of the lens of the eye”, “Cloudy lens”, “Lens
opacities”, “Lens opacity”)
</td>
<td style="text-align:left;">
c(“biolink:BiologicalEntity”, “biolink:DiseaseOrPhenotypicFeature”,
“biolink:Entity”, “biolink:NamedThing”, “biolink:PhenotypicFeature”,
“biolink:ThingWithTaxon”)
</td>
<td style="text-align:left;">
“<a href="http://purl.obolibrary.org/obo/HP_0000518" class="external-link uri">http://purl.obolibrary.org/obo/HP_0000518</a>”
</td>
<td style="text-align:left;">
c(“Fyler:4865”, “MSH:D002386”, “SNOMEDCT_US:128306009”,
“SNOMEDCT_US:193570009”, “SNOMEDCT_US:247053007”, “UMLS:C0086543”,
“UMLS:C1510497”)
</td>
<td style="text-align:left;">
“HP”
</td>
<td style="text-align:left;">
“phenio_nodes”
</td>
</tr>
</tbody>
</table>
</div>
</details><details><summary><b>Edge Data</b>
</summary><p>
Showing 11 of 11 edges:
</p>
<div style="font-size: 0.8em; max-height: 400px;overflow-y: auto;border-left: 1px solid #ddd;border-right:  1px solid #ddd;border-bottom: 1px solid #ddd;">
<table class="table table table-striped table-hover table-condensed" style="margin-left: auto; margin-right: auto;">
<thead><tr>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
from
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
to
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
subject
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
predicate
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
object
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
metric
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
score
</th>
<th style="text-align:left;position: sticky; top:0; background-color: #FFFFFF;">
ancestor_id
</th>
</tr></thead>
<tbody>
<tr>
<td style="text-align:left;">
9
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
“HP:0000938”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0005136”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
6.20804739894599
</td>
<td style="text-align:left;">
“HP:0011842”
</td>
</tr>
<tr>
<td style="text-align:left;">
2
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
“HP:0000974”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0001377”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
4.12905180500894
</td>
<td style="text-align:left;">
“HP:0000118”
</td>
</tr>
<tr>
<td style="text-align:left;">
3
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
“HP:0001027”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0001377”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
4.12905180500894
</td>
<td style="text-align:left;">
“HP:0000118”
</td>
</tr>
<tr>
<td style="text-align:left;">
4
</td>
<td style="text-align:left;">
18
</td>
<td style="text-align:left;">
“HP:0001030”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0000518”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
4.12905180500894
</td>
<td style="text-align:left;">
“HP:0000118”
</td>
</tr>
<tr>
<td style="text-align:left;">
10
</td>
<td style="text-align:left;">
14
</td>
<td style="text-align:left;">
“HP:0001058”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0000486”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
4.12905180500894
</td>
<td style="text-align:left;">
“HP:0000118”
</td>
</tr>
<tr>
<td style="text-align:left;">
5
</td>
<td style="text-align:left;">
16
</td>
<td style="text-align:left;">
“HP:0001065”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0001371”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
5.6587800363266
</td>
<td style="text-align:left;">
“UPHENO:0012541”
</td>
</tr>
<tr>
<td style="text-align:left;">
6
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
“HP:0001073”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0005136”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
9.87469715486756
</td>
<td style="text-align:left;">
“HP:0003549”
</td>
</tr>
<tr>
<td style="text-align:left;">
7
</td>
<td style="text-align:left;">
15
</td>
<td style="text-align:left;">
“HP:0001075”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0005136”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
9.87469715486756
</td>
<td style="text-align:left;">
“HP:0003549”
</td>
</tr>
<tr>
<td style="text-align:left;">
11
</td>
<td style="text-align:left;">
17
</td>
<td style="text-align:left;">
“HP:0001252”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0003199”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
8.69625474174526
</td>
<td style="text-align:left;">
“HP:0003011”
</td>
</tr>
<tr>
<td style="text-align:left;">
8
</td>
<td style="text-align:left;">
13
</td>
<td style="text-align:left;">
“HP:0002761”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“HP:0001377”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
9.98265640300348
</td>
<td style="text-align:left;">
“HP:0011729”
</td>
</tr>
<tr>
<td style="text-align:left;">
1
</td>
<td style="text-align:left;">
12
</td>
<td style="text-align:left;">
“MONDO:0007522”
</td>
<td style="text-align:left;">
“computed:best_matches”
</td>
<td style="text-align:left;">
“MONDO:0007947”
</td>
<td style="text-align:left;">
“ancestor_information_content”
</td>
<td style="text-align:left;">
8.88604807920409
</td>
<td style="text-align:left;">
“MONDO:0000426”
</td>
</tr>
</tbody>
</table>
</div>
</details><p><br></p>
<p>Note that the Disease nodes were included in the match as well.
Although not displayed by <code><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot()</a></code>, edges also include the
metric, similarity score, and ID of the least common ancestor relating
the nodes.</p>
<p>We can use <code>tidygraph</code>’s <code><a href="https://tidygraph.data-imaginist.com/reference/graph_join.html" class="external-link">graph_join()</a></code> to see
this result in the context of the original query and target graphs.</p>
<div class="sourceCode" id="cb17"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span><span class="va">eds_phenos</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://tidygraph.data-imaginist.com/reference/graph_join.html" class="external-link">graph_join</a></span><span class="op">(</span><span class="va">marfan_phenos</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://tidygraph.data-imaginist.com/reference/graph_join.html" class="external-link">graph_join</a></span><span class="op">(</span><span class="va">sim</span><span class="op">)</span> <span class="op">|&gt;</span></span>
<span>    <span class="fu"><a href="https://rdrr.io/r/graphics/plot.default.html" class="external-link">plot</a></span><span class="op">(</span><span class="op">)</span></span></code></pre></div>
<p><img src="monarchr_files/figure-html/unnamed-chunk-14-1.png" width="960"></p>
</div>
</div>
<div class="footnotes footnotes-end-of-document">
<hr>
<ol>
<li id="fn1"><p>The <code>%in_list%</code> and other operators work the
same for <code><a href="../reference/file_engine.html">file_engine()</a></code>s, which store multi-valued
properties in list columns where <code>%in%</code> does not provide the
desired semantics. Note that <code>%in_list%</code> can only take a
length-one character vector as the left-hand side, and the right hand
side must be a single node property name in the KG.<a href="#fnref1" class="footnote-back">↩︎</a></p></li>
<li id="fn2"><p>The <code>limit</code> parameter for
<code><a href="../reference/expand.html">expand()</a></code> is more complex than for
<code><a href="../reference/fetch_nodes.html">fetch_nodes()</a></code>, because the query graph also contains edges
which may be part of the neighborhood that would be fetched;
<code><a href="../reference/expand.html">expand()</a></code> attempts to fetch <code>limit</code> new edges
that are not part of the query already. Further, unlike in
<code><a href="../reference/fetch_nodes.html">fetch_nodes()</a></code> where nodes are ordered by ID, edges returned
by <code><a href="../reference/expand.html">expand()</a></code> have no explicit order, and results with
<code>limit</code> should not be considered reproducible or
representative.<a href="#fnref2" class="footnote-back">↩︎</a></p></li>
</ol>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">

      </div>

</div>



      <footer><div class="copyright">
  <p></p>
<p>Developed by Shawn O’Neil, Brian Schilder.</p>
</div>

<div class="pkgdown">
  <p></p>
<p>Site built with <a href="https://pkgdown.r-lib.org/" class="external-link">pkgdown</a> 2.1.0.</p>
</div>

      </footer>
</div>






  </body>
</html>
